roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8740106?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8740106?buildTab=artifacts#/activerecord) on master @ [7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260](https://github.com/cockroachdb/cockroach/commits/7e2df35a2f6bf7a859bb0539c8ca43c4e72ed260):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:201).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.2-405-g7e2df35a2f
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Encryption::StoragePerformanceTest#test_storage_overload_without_storing_keys_is_acceptable (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
/cc @cockroachdb/sql-sessions
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-24607

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8750108?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8750108?buildTab=artifacts#/activerecord) on master @ [c95bef097bd4c213c6b5c0c125a9a846c4479d73](https://github.com/cockroachdb/cockroach/commits/c95bef097bd4c213c6b5c0c125a9a846c4479d73):


```
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_automatically_save_the_associated_models (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_rollback_any_changes_if_an_exception_occurred_while_saving (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_sum_uses_calculations (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_not_update_children_when_parent_creation_with_no_reason (unexpected)
--- FAIL: RelationTest#test_reorder_with_first (unexpected)
--- FAIL: FinderTest#test_find_on_relation_with_large_number (unexpected)
--- FAIL: RelationTest#test_scoped_build (unexpected)
--- FAIL: QueryCacheTest#test_query_cache_is_enabled_on_all_connection_pools (unexpected)
--- FAIL: FinderTest#test_joins_dont_clobber_id (unexpected)
--- FAIL: RelationTest#test_empty (unexpected)
--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_conditions_on_nested_association (unexpected)
--- FAIL: RelationTest#test_automatically_added_reorder_references (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_ignore_new_associated_records_with_truthy_destroy_attribute (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_with_table_name_and_empty_hash (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_with_boolean_for_string_column (unexpected)
--- FAIL: BelongsToAssociationsTest#test_optional_relation (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_has_one_reflection (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_and_removed_parent (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_find_all (unexpected)
--- FAIL: AssociationsTest#test_eager_loading_should_not_change_count_of_children (unexpected)
--- FAIL: RelationTest#test_finding_with_order_concatenated (unexpected)
--- FAIL: EagerAssociationTest#test_include_has_one_using_primary_key (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_run_add_callback_procs_for_habtm (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_many_with_has_and_belongs_to_many_source_reflection_preload_via_joins (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_scope_on_polymorphic_reflection (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_rollback_destructions_if_an_exception_occurred_while_saving_a_parent (unexpected)
--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_timestamps_doesnt_set_precision_on_create_table (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through_with_nonstandard_id (unexpected)
--- FAIL: RelationTest#test_order_triggers_eager_loading_when_ordering_using_symbols (unexpected)
--- FAIL: PostGISTest#test_save_and_load_point (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_save_bang_should_raise_exception_when_join_record_has_errors (unexpected)
--- FAIL: BelongsToAssociationsTest#test_with_different_class_name (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_can_use_symbols_as_object_identifier (unexpected)
--- FAIL: RelationTest#test_order_triggers_eager_loading (unexpected)
--- FAIL: BelongsToAssociationsTest#test_assigning_an_association_doesn't_result_in_duplicate_objects (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_skip_validation_on_has_many_if_destroyed (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_ordered_has_many_through (unexpected)
--- FAIL: AssociationsExtensionsTest#test_named_two_extensions_on_habtm (unexpected)
--- FAIL: BelongsToAssociationsTest#test_dangerous_association_name_raises_ArgumentError (unexpected)
--- FAIL: ReflectionTest#test_column_string_type_and_limit (unexpected)
--- FAIL: MultiParameterAttributeTest#test_multiparameter_assignment_of_aggregation (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_string_attribute (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_boolean_value_true (unexpected)
--- FAIL: ReflectionTest#test_reflections_should_return_keys_as_strings (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_default_invalid_error_from_i18n (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_polymorphic_with_rewhere (unexpected)
--- FAIL: MultipleDbTest#test_transactions_across_databases (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_not_save_and_return_false_if_a_callback_cancelled_saving_in_either_create_or_update (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8763696?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8763696?buildTab=artifacts#/activerecord) on master @ [e9c96e7179e19aae2f8d386f67eb950db8c3354b](https://github.com/cockroachdb/cockroach/commits/e9c96e7179e19aae2f8d386f67eb950db8c3354b):


```
--- FAIL: HasManyAssociationsTest#test_associations_order_should_be_priority_over_throughs_order (unexpected)
--- FAIL: FinderTest#test_#skip_query_cache!_for_#exists?_with_a_limited_eager_load (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all_on_association_clears_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_add_record_to_collection_should_change_its_updated_at (unexpected)
--- FAIL: SpatialQueriesTest#test_query_rgeo_feature_node (unexpected)
--- FAIL: HasManyAssociationsTest#test_attributes_are_being_set_when_initialized_from_has_many_association_with_multiple_where_clauses (unexpected)
--- FAIL: EagerAssociationTest#test_eager-loading_with_a_polymorphic_association_won't_work_consistently (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- FAIL: EagerAssociationTest#test_preloading_the_same_association_twice_works (unexpected)
--- FAIL: EagerAssociationTest#test_including_association_based_on_sql_condition_and_no_database_column (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_2#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: FinderTest#test_member_on_unloaded_relation_with_limit (unexpected)
--- FAIL: AssociationProxyTest#test_proxy_object_can_be_stubbed (unexpected)
--- FAIL: AssociationProxyTest#test_target_merging_ignores_persisted_in_memory_records (unexpected)
--- FAIL: RelationScopingTest#test_scope_breaks_caching_on_collections (unexpected)
--- FAIL: EagerAssociationTest#test_works_in_combination_with_order(:symbol)_and_reorder(:symbol) (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all_association_with_primary_key_deletes_correct_records (unexpected)
--- FAIL: ForeignKeyFixturesTest#test_number2 (unexpected)
--- FAIL: FinderTest#test_dynamic_finder_on_one_attribute_with_conditions_returns_same_results_after_caching (unexpected)
--- FAIL: EagerAssociationTest#test_deep_including_through_habtm (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_on_association_clears_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_counting (unexpected)
--- FAIL: HasManyAssociationsTest#test_include_method_in_has_many_association_should_return_true_for_instance_added_with_build (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_first_nth_or_last_on_existing_record_with_build_should_load_association (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_none_should_return_false_if_any (unexpected)
--- FAIL: AssociationsTest#test_should_construct_new_finder_sql_after_create (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_second_bang (unexpected)
--- FAIL: FinderTest#test_sole_failing_none (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_dependent_when_deleted_from_association (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_first (unexpected)
--- FAIL: SpatialQueriesTest#test_query_point_wkt (unexpected)
--- FAIL: HasManyAssociationsTest#test_get_ids (unexpected)
--- FAIL: FinderTest#test_find_with_ids_with_no_id_passed (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_empty_array_with_aggregate_having_multiple_mappings (unexpected)
--- FAIL: EagerAssociationTest#test_eager_loading_of_instance_dependent_associations_is_not_supported (unexpected)
--- FAIL: PreloaderTest#test_some_already_loaded_associations (unexpected)
--- FAIL: FinderTest#test_last_bang_present (unexpected)
--- FAIL: FinderTest#test_select_value (unexpected)
--- FAIL: EagerAssociationTest#test_including_duplicate_objects_from_has_many (unexpected)
--- FAIL: EagerAssociationTest#test_loading_association_with_string_joins (unexpected)
--- FAIL: AssociationProxyTest#test_reload_returns_association (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V6_0#test_change_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: HasManyAssociationsTest#test_regular_create_on_has_many_when_parent_is_new_raises (unexpected)
--- FAIL: FinderTest#test_fifth_have_primary_key_order_by_default (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_many (unexpected)
--- FAIL: FinderTest#test_exists_with_polymorphic_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8803571?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8803571?buildTab=artifacts#/activerecord) on master @ [e028ce5b14505dfd17ef8b13001c0ab8ac811e3c](https://github.com/cockroachdb/cockroach/commits/e028ce5b14505dfd17ef8b13001c0ab8ac811e3c):


```
--- FAIL: HasManyAssociationsTest#test_dynamic_find_should_respect_association_order (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all,_when_not_loaded,_doesn't_load_the_records (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_scoped_to_defining_class (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_ids (unexpected)
--- FAIL: HasManyAssociationsTest#test_adding (unexpected)
--- FAIL: HasManyAssociationsTest#test_create (unexpected)
--- FAIL: HasManyAssociationsTest#test_clearing_updates_counter_cache (unexpected)
--- FAIL: HasManyAssociationsTest#test_finding_using_primary_key (unexpected)
--- FAIL: HasManyAssociationsTest#test_association_proxy_transaction_method_starts_transaction_in_association_class (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_by_integer_id (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_from_association_with_nil_values_should_work (unexpected)
--- FAIL: HasManyAssociationsTest#test_collection_empty_with_dirty_target (unexpected)
--- FAIL: HasManyAssociationsTest#test_building_the_associated_object_with_sti_subclass (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_the_association_with_an_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_and_create_from_association_should_respect_passed_attributes_over_default_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_followed_by_save_does_not_load_target (unexpected)
--- FAIL: HasManyAssociationsTest#test_ids_reader_cache_should_be_cleared_when_collection_is_deleted (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_immutable_strings_by_default_does_not_affect_``attribute_:foo,_:string`` (unexpected)
--- FAIL: HasManyAssociationsTest#test_loading_association_in_validate_callback_doesnt_affect_persistence (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_from_association_sets_inverse_instance (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_are_memoized_before_type_casting (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: HasManyAssociationsTest#test_get_ids_for_unloaded_associations_does_not_load_them (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all_on_association_is_the_same_as_not_loaded (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroying_by_integer_id (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_with_polymorphic_has_many_does_not_allow_to_override_type_and_id (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_the_select_one_on_the_same_table (unexpected)
--- FAIL: HasManyAssociationsTest#test_replace_failure (unexpected)
--- FAIL: HasManyAssociationsTest#test_association_size_calculation_works_with_default_scoped_selects_when_not_previously_fetched (unexpected)
--- FAIL: HasManyAssociationsTest#test_sending_new_to_association_proxy_should_have_same_effect_as_calling_new (unexpected)
--- FAIL: HasManyAssociationsTest#test_in_memory_replacements_do_not_execute_callbacks (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_case_sensitive_uniqueness (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache_without_dependent_option (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_limit_and_utf8 (unexpected)
--- FAIL: HasManyAssociationsTest#test_to_a_should_dup_target (unexpected)
--- FAIL: HasManyAssociationsTest#test_association_with_instance_dependent_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_anonymous_has_many (unexpected)
--- FAIL: HasManyAssociationsTest#test_associations_autosaves_when_object_is_already_persisted (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_changing_defaults (unexpected)
--- FAIL: HasManyAssociationsTest#test_clearing_without_initial_access (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_many_via_block (unexpected)
--- FAIL: HasManyAssociationsTest#test_exists_respects_association_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_via_block (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_overloaded_properties_save (unexpected)
--- FAIL: HasManyAssociationsTest#test_include_returns_false_for_non_matching_record_to_verify_scoping (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_without_counter_cache_option (unexpected)
--- FAIL: HasManyAssociationsTest#test_include_uses_array_include_after_loaded (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8880604?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8880604?buildTab=artifacts#/activerecord) on master @ [21786aa112e6b822858f281c1cc59608987c5c0a](https://github.com/cockroachdb/cockroach/commits/21786aa112e6b822858f281c1cc59608987c5c0a):


```
--- FAIL: BelongsToAssociationsTest#test_optional_relation (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_belongs_to (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_collection_size_uses_counter_cache_if_it_exists (unexpected)
--- FAIL: PostGISTest#test_set_and_get_point_from_wkt (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_primary_key (unexpected)
--- FAIL: InsertAllTest#test_insert_all_and_upsert_all_works_with_composite_primary_keys_when_unique_by_is_provided (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_with_touch:_:written_on (unexpected)
--- FAIL: AssociationsJoinModelTest#test_belongs_to_shared_parent (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_string_argument (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_polymorphic_has_one_with_destroy (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_by_default_can_be_set_per_model (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_explicit_sti_base_class (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
--- FAIL: BelongsToAssociationsTest#test_attributes_are_set_without_error_when_initialized_from_belongs_to_association_with_array_in_where_clause (unexpected)
--- FAIL: CounterCacheTest#test_update_counter_with_initial_null_value (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_proxy_should_respond_to_private_methods_via_send (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_polymorphic_has_many (unexpected)
--- FAIL: CounterCacheTest#test_decrement_counters_with_touch:_true (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_unloaded_relation_methods_if_strict_loading_by_default (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_id_assigning (unexpected)
--- FAIL: StrictLoadingTest#test_preload_audit_logs_are_strict_loading_because_it_is_strict_loading_by_default (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_create_model_with_inheritance_and_custom_base_class (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
--- FAIL: ForeignKeyFixturesTest#test_number1 (unexpected)
--- FAIL: PostGISTest#test_custom_column (unexpected)
--- FAIL: ReadOnlyTest#test_has_many_with_through_is_not_implicitly_marked_readonly_while_finding_last (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_create_model_with_inheritance (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_does_call_updated_at_when_it_is_assigned_via_a_string (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_association_class (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_column (unexpected)
--- FAIL: BelongsToAssociationsTest#test_creating_the_belonging_object_with_primary_key (unexpected)
--- FAIL: AssociationsJoinModelTest#test_proper_error_message_for_eager_load_and_includes_association_errors (unexpected)
--- FAIL: CounterCacheTest#test_increment_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_key_has_no_version_when_versioning_is_on (unexpected)
--- FAIL: AssociationsJoinModelTest#test_associating_unsaved_records_with_has_many_through (unexpected)
--- FAIL: AssociationsJoinModelTest#test_calculations_on_has_many_through_should_disambiguate_fields (unexpected)
--- FAIL: BelongsToAssociationsTest#test_clearing_an_association_clears_the_associations_inverse (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_polymorphic_has_many_with_delete_all (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_polymorphic_has_one_with_nullify (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_empty_update (unexpected)
--- FAIL: ModulesTest#test_assign_ids (unexpected)
--- FAIL: AssociationsJoinModelTest#test_include_polymorphic_has_one (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_implicit_sti_base_class (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_with_custom_primary_key_on_belongs_to_source (unexpected)
--- FAIL: DDLTest#test_array_columns (unexpected)
--- FAIL: BelongsToAssociationsTest#test_natural_assignment_to_nil (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8965205?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/8965205?buildTab=artifacts#/activerecord) on master @ [1b162d1b274eec7b307fbbfca7294460bfdef025](https://github.com/cockroachdb/cockroach/commits/1b162d1b274eec7b307fbbfca7294460bfdef025):


```
--- FAIL: ReflectionTest#test_scope_chain_of_polymorphic_association_does_not_leak_into_other_hmt_associations (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_adding_a_collection (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_string_primary_key_settable_before_save (unexpected)
--- FAIL: PrimaryKeysTest#test_integer_key (unexpected)
--- FAIL: PrimaryKeysTest#test_find_with_multiple_ids_should_quote_pkey (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_column_is_mass_assignable (unexpected)
--- FAIL: FixturesTest#test_no_args_returns_all (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_with_extend_option (unexpected)
--- FAIL: FixturesTest#test_binary_in_fixtures (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_doesn't_touch_timestamps_with_touch:_[] (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_with_invalid_value (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many_in_a_namespaced_model_pointing_to_a_namespaced_model (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_deleting (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_distinct_after_the_fact (unexpected)
--- FAIL: ReflectionTest#test_has_many_reflection (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_not_polymorphic_id_and_type_as_nand (unexpected)
--- FAIL: ReadOnlyTest#test_has_many_with_through_is_not_implicitly_marked_readonly_while_finding_last (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_polymorphic_nested_array_where_not (unexpected)
--- FAIL: ReflectionTest#test_non_existent_types_are_identity_types (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_with_table_name_and_empty_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
--- FAIL: FixturesTest#test_yaml_file_with_invalid_column (unexpected)
--- FAIL: ReflectionTest#test_read_attribute_names (unexpected)
--- FAIL: ReflectionTest#test_content_columns (unexpected)
--- FAIL: OptimisticLockingTest#test_update_with_lock_version_without_default_should_work_on_dirty_value_before_type_cast (unexpected)
--- FAIL: ReflectionTest#test_join_table_can_be_overridden (unexpected)
--- FAIL: SerializedAttributeTest#test_json_read_db_null (unexpected)
--- FAIL: FixturesTest#test_logger_level_invariant (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_invalid_foreign_key (unexpected)
--- FAIL: CounterCacheTest#test_reset_counter_of_has_many_:through_association (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_with_table_name_and_empty_hash (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_attributes_are_being_set_when_initialized_from_habtm_association_with_where_clause (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_increment_counter_updates_custom_lock_version (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_json_attribute_returns_unserialized_value (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_scoped_find_on_through_association_doesnt_return_read_only_records (unexpected)
--- FAIL: ReflectionTest#test_scope_chain_does_not_interfere_with_hmt_with_polymorphic_case_and_sti (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_copies_bind_params_in_the_right_order (unexpected)
--- FAIL: ReadOnlyTest#test_has_many_with_through_is_not_implicitly_marked_readonly_while_finding_first (unexpected)
--- FAIL: CounterCacheTest#test_update_multiple_counters_with_touch:_:written_on (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many_is_usable_with_belongs_to_required_by_default (unexpected)
--- FAIL: CounterCacheTest#test_increment_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_default_class (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_with_duration_for_string_column (unexpected)
--- FAIL: ReadOnlyTest#test_readonly_scoping (unexpected)
--- FAIL: PrimaryKeysTest#test_id_column_that_is_not_primary_key (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_symbol_join_table (unexpected)
--- FAIL: FixturesTest#test_empty_yaml_fixture (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9088337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9088337?buildTab=artifacts#/activerecord) on master @ [d68f5cdfb25476aaf874bff0bb6515eba02efcd2](https://github.com/cockroachdb/cockroach/commits/d68f5cdfb25476aaf874bff0bb6515eba02efcd2):


```
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_constraint_by_name (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_first_after_reset_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_without_loading_association (unexpected)
--- FAIL: FinderTest#test_find_with_bad_sql (unexpected)
--- FAIL: HasManyAssociationsTest#test_include_checks_if_record_exists_if_target_not_loaded (unexpected)
--- FAIL: FilterAttributesTest#test_filter_attributes_on_pretty_print_should_handle_[FILTERED]_value_properly (unexpected)
--- FAIL: HasManyAssociationsTest#test_finding_with_condition_hash (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_first_nth_or_last_on_new_record_should_not_run_queries (unexpected)
--- FAIL: FinderTest#test_take_bang_missing (unexpected)
--- FAIL: FinderTest#test_select_values (unexpected)
--- FAIL: FinderTest#test_find_by_with_associations (unexpected)
--- FAIL: HasManyAssociationsTest#test_with_polymorphic_has_many_with_custom_columns_name (unexpected)
--- FAIL: FinderTest#test_find_on_association_proxy_conditions (unexpected)
--- FAIL: HasManyAssociationsTest#test_attributes_are_being_set_when_initialized_from_has_many_association_with_where_clause (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_third_bang (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_one_should_return_true_if_one (unexpected)
--- FAIL: FinderTest#test_first_bang_present (unexpected)
--- FAIL: HasManyAssociationsTest#test_associations_order_should_be_priority_over_throughs_order (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_many_should_count_instead_of_loading_association (unexpected)
--- FAIL: FinderTest#test_find_by_with_large_number (unexpected)
--- FAIL: HasManyAssociationsTest#test_depends_and_nullify (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_unabstracted_database_dependent_types (unexpected)
--- FAIL: FinderTest#test_exists_with_joins (unexpected)
--- FAIL: CockroachDB::FixturesResetPkSequenceTest#test_create_fixtures_resets_sequences_when_not_cached (unexpected)
--- FAIL: FinderTest#test_exists_with_distinct_association_includes_and_limit (unexpected)
--- FAIL: RelationScopingTest#test_scoping_respects_sti_constraint (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_does_not_raise_when_association_errors_on_destroy (unexpected)
--- FAIL: HasManyAssociationsTest#test_association_with_extend_option_with_multiple_extensions (unexpected)
--- FAIL: HasManyAssociationsTest#test_counter_cache_updates_in_memory_after_create_with_array (unexpected)
--- FAIL: RelationScopingTest#test_delete_all_default_scope_filters_on_joins (unexpected)
--- FAIL: FinderTest#test_third_have_primary_key_order_by_default (unexpected)
--- FAIL: FinderTest#test_find_ignores_previously_inserted_record (unexpected)
--- FAIL: FinderTest#test_exists_with_aggregate_having_three_mappings_with_one_difference (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
--- FAIL: FinderTest#test_fourth (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: FinderTest#test_find_with_hash_conditions_on_joined_table_and_with_range (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_followed_by_save_does_not_load_target (unexpected)
--- FAIL: FinderTest#test_find_by!_with_multi-arg_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: SuppressorTest#test_suppresses_create_in_callback (unexpected)
--- FAIL: SecureTokenTest#test_regenerating_the_secure_token (unexpected)
--- FAIL: FinderTest#test_third_with_offset (unexpected)
--- FAIL: HasManyAssociationsTest#test_joins_with_namespaced_model_should_use_correct_type (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_second_to_last_bang (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_one_on_loaded_association_should_not_use_query (unexpected)
--- FAIL: FinderTest#test_exists_with_nil_arg (unexpected)
--- FAIL: FinderTest#test_exists_with_polymorphic_relation (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9106835?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9106835?buildTab=artifacts#/activerecord) on master @ [f0489334a0ee6980a9d365b361d2fce4b2cdc05b](https://github.com/cockroachdb/cockroach/commits/f0489334a0ee6980a9d365b361d2fce4b2cdc05b):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.7-dev-f0489334a0ee6980a9d365b361d2fce4b2cdc05b
Tests run against activerecord 7.0.3
7163 Total Tests Run
7152 tests passed
11 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_update_children_when_autosave_is_true_and_parent_is_new_but_child_is_not (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_not_update_children_when_parent_creation_with_no_reason (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_not_save_and_return_false_if_a_callback_cancelled_saving_in_either_create_or_update (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9130968?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9130968?buildTab=artifacts#/activerecord) on master @ [53dbb86acb1d48309530181b94838faf937084d3](https://github.com/cockroachdb/cockroach/commits/53dbb86acb1d48309530181b94838faf937084d3):


```
--- FAIL: HasManyAssociationsTest#test_find_one_message_on_primary_key (unexpected)
--- FAIL: HasManyAssociationsTest#test_build (unexpected)
--- FAIL: HasOneAssociationsTest#test_clearing_an_association_clears_the_associations_inverse (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_associations_on_new_records_use_null_relations (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_calling_dup_method (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_with_primary_key_updates_foreign_id_field_for_new_and_saved_records (unexpected)
--- FAIL: BelongsToAssociationsTest#test_where_with_custom_primary_key (unexpected)
--- FAIL: HasManyAssociationsTest#test_dynamic_find_should_respect_association_order_for_through (unexpected)
--- FAIL: HasOneAssociationsTest#test_building_the_associated_object_with_implicit_sti_base_class (unexpected)
--- FAIL: HasOneAssociationsTest#test_creation_failure_due_to_new_record_should_raise_error (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_grouped (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_and_removed_parent (unexpected)
--- FAIL: HasOneAssociationsTest#test_nullify_on_polymorphic_association (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_association_with_bang_failing (unexpected)
--- FAIL: HasManyAssociationsTest#test_finder_bang_method_with_dirty_target (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_and_create_should_not_happen_within_scope (unexpected)
--- FAIL: BelongsToAssociationsTest#test_clearing_an_association_clears_the_associations_inverse (unexpected)
--- FAIL: BelongsToAssociationsTest#test_multiple_counter_cache_with_after_create_update (unexpected)
--- FAIL: BelongsToAssociationsTest#test_missing_attribute_error_is_raised_when_no_foreign_key_attribute (unexpected)
--- FAIL: BelongsToAssociationsTest#test_id_assignment (unexpected)
--- FAIL: SerializedAttributeTest#test_json_read_db_null (unexpected)
--- FAIL: UniquenessValidationTest#test_validates_uniqueness_with_validates (unexpected)
--- FAIL: HasManyAssociationsTest#test_restrict_with_error_with_locale (unexpected)
--- FAIL: HasOneAssociationsTest#test_association_change_calls_destroy (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_does_not_use_order_by (unexpected)
--- FAIL: HasManyAssociationsTest#test_defining_has_many_association_with_nullify_dependency_lazily_evaluates_target_class (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_counter_cache (unexpected)
--- FAIL: CounterCacheTest#test_increment_counter (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_multiple_clean_fixtures (unexpected)
--- FAIL: DDLTest#test_create_simple_geometry (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_with_default (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_in_base_class (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all_with_option_delete_all (unexpected)
--- FAIL: DDLTest#test_non_spatial_column_limits (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_assignment_triggers_save_on_change_on_replacing_object (unexpected)
--- FAIL: HasManyAssociationsTest#test_collection_not_empty_after_building (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_average_interval_type (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_respects_hash_condition (unexpected)
--- FAIL: CounterCacheTest#test_increment_counters_with_touch:_true (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_postgresql_interval_parser (unexpected)
--- FAIL: CounterCacheTest#test_reset_multiple_counters (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_without_counter_cache_option (unexpected)
--- FAIL: HasOneAssociationsTest#test_destroyed_by_association_set_in_child_destroy_callback_on_replace (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_followed_by_save_does_not_load_target (unexpected)
--- FAIL: HasManyAssociationsTest#test_to_a_should_dup_target (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_autosave_with_primary_key_manually_set (unexpected)
--- FAIL: CounterCacheTest#test_update_multiple_counters_with_touch:_true (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9182736?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9182736?buildTab=artifacts#/activerecord) on master @ [78bb7696847e1d8abf5d6a199c6dd3414a568691](https://github.com/cockroachdb/cockroach/commits/78bb7696847e1d8abf5d6a199c6dd3414a568691):


```
47 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
39 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: AssociationProxyTest#test_getting_a_scope_from_an_association (unexpected)
--- FAIL: AssociationProxyTest#test_load_does_load_target (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_change_table (unexpected)
--- FAIL: AssociationProxyTest#test_push_followed_by_save_does_not_load_target (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_stored_column (unexpected)
--- FAIL: AssociationProxyTest#test_proxy_object_is_cached (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column (unexpected)
--- FAIL: AssociationProxyTest#test_inspect_does_not_reload_a_not_yet_loaded_target (unexpected)
--- FAIL: AssociationProxyTest#test_save_on_parent_saves_children (unexpected)
--- FAIL: AssociationProxyTest#test_prepend_is_not_defined (unexpected)
--- FAIL: AssociationProxyTest#test_append_behaves_like_push (unexpected)
--- FAIL: AssociationProxyTest#test_reset_unloads_target (unexpected)
--- FAIL: PostgresqlNumberTest#test_data_type (unexpected)
--- FAIL: AssociationProxyTest#test_create_with_bang_via_association_with_block (unexpected)
--- FAIL: TestAutosaveAssociationsInGeneral#test_cyclic_autosaves_do_not_add_multiple_validations (unexpected)
--- FAIL: AssociationProxyTest#test_save_on_parent_does_not_load_target (unexpected)
--- FAIL: AssociationProxyTest#test_inverses_get_set_of_subsets_of_the_association (unexpected)
--- FAIL: AssociationProxyTest#test_target_merging_ignores_persisted_in_memory_records (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_suffix (unexpected)
--- FAIL: SerializationTest#test_find_records_by_serialized_attributes_through_join (unexpected)
--- FAIL: AssociationProxyTest#test_push_does_not_load_target (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_non_persisted_column (unexpected)
--- FAIL: AssociationProxyTest#test_first!_works_on_loaded_associations (unexpected)
--- FAIL: AssociationProxyTest#test_reload_returns_association (unexpected)
--- FAIL: PostgresqlIntegerTest#test_schema_properly_respects_bigint_ranges (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_prefix (unexpected)
--- FAIL: PostgresqlNumberTest#test_update (unexpected)
--- FAIL: AssociationProxyTest#test_pick_uses_loaded_target (unexpected)
--- FAIL: ActiveRecord::Migration::ChangeSchemaWithDependentObjectsTest#test_create_table_with_force_cascade_drops_dependent_objects (unexpected)
--- FAIL: AssociationProxyTest#test_create_via_association_with_block (unexpected)
--- FAIL: AssociationProxyTest#test_push_does_not_lose_additions_to_new_record (unexpected)
--- FAIL: AssociationProxyTest#test_proxy_association_accessor (unexpected)
--- FAIL: AssociationProxyTest#test_proxy_object_can_be_stubbed (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: AssociationProxyTest#test_push_has_many_through_does_not_load_target (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnAHasManyAssociationWithAcceptsNestedAttributes#test_errors_details_should_be_indexed_when_passed_as_array (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnAHasManyAssociationWithAcceptsNestedAttributes#test_errors_should_be_indexed_when_passed_as_array (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column_with_full_inserts (unexpected)
--- FAIL: AssociationProxyTest#test_scoped_allows_conditions (unexpected)
--- FAIL: AssociationProxyTest#test_pluck_uses_loaded_target (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9225780?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9225780?buildTab=artifacts#/activerecord) on master @ [19a6b804d3aff74d74619a75cac3b52338c7aa02](https://github.com/cockroachdb/cockroach/commits/19a6b804d3aff74d74619a75cac3b52338c7aa02):


```
47 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
39 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_constraint_by_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_nullify_foreign_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_not_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_restrict_foreign_key (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_non_persisted_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: ActiveRecord::Encryption::StoragePerformanceTest#test_storage_overload_without_storing_keys_is_acceptable (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
--- FAIL: TimestampsWithoutTransactionTest#test_do_not_write_timestamps_on_save_if_they_are_not_attributes (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column_with_full_inserts (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_not_set (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_infers_column (unexpected)
--- FAIL: CockroachDB::MigrationTest#test_remove_column_with_if_not_exists_not_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_symbol_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_invalid_foreign_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_symbol_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_stored_column (unexpected)
--- FAIL: MigratorTest#test_migrations_status_with_schema_define_in_subdirectories (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_the_select_one_on_the_same_table (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_column (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_change_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9353426?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9353426?buildTab=artifacts#/activerecord) on master @ [41581c367b3a1d7893c2945465c57ffa841035e3](https://github.com/cockroachdb/cockroach/commits/41581c367b3a1d7893c2945465c57ffa841035e3):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-41581c367b3a1d7893c2945465c57ffa841035e3
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionHandlersMultiDbTest#test_switching_connections_with_database_config_hash (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9353437?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9353437?buildTab=artifacts#/activerecord) on master @ [41581c367b3a1d7893c2945465c57ffa841035e3](https://github.com/cockroachdb/cockroach/commits/41581c367b3a1d7893c2945465c57ffa841035e3):


```
--- FAIL: AssociationProxyTest#test_save_on_parent_does_not_load_target (unexpected)
--- FAIL: RelationTest#test_find_all_using_where_with_relation_with_joins (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_on_habtm (unexpected)
--- FAIL: MigratorTest#test_migrations_status_with_schema_define_in_subdirectories (unexpected)
--- FAIL: RequiredAssociationsTest#test_required_belongs_to_associations_have_presence_validated (unexpected)
--- FAIL: FinderTest#test_take (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
--- FAIL: RelationTest#test_reverse_order_with_reorder_nil_removes_the_order (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_not_remove_scheduled_destroys_when_loading_association (unexpected)
--- FAIL: FinderTest#test_include_on_unloaded_relation_without_match (unexpected)
--- FAIL: AssociationsJoinModelTest#test_inherited_has_many (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_save_new_record_that_has_same_value_as_existing_record_marked_for_destruction_on_field_that_has_unique_index (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_fifth_bang (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: RelationTest#test_find_all_with_multiple_should_use_and (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_explicit_table_name_and_aggregate (unexpected)
--- FAIL: AssociationsJoinModelTest#test_create_polymorphic_has_many_with_scope (unexpected)
--- FAIL: RelationTest#test_finding_with_order_limit_and_offset (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_marshal_dump (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many (unexpected)
--- FAIL: ModulesTest#test_eager_loading_in_modules (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_has_many_sti (unexpected)
--- FAIL: DelegatedTypeTest#test_delegated_type_predicates (unexpected)
--- FAIL: FinderTest#test_find_by_on_relation_with_large_number (unexpected)
--- FAIL: ModulesTest#test_module_table_name_suffix_with_global_suffix (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_belongs_to (unexpected)
--- FAIL: RelationTest#test_extracted_association (unexpected)
--- FAIL: RelationTest#test_create_or_find_by (unexpected)
--- FAIL: RelationTest#test_size_with_zero_limit (unexpected)
--- FAIL: AssociationProxyTest#test_append_behaves_like_push (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_add_remove_single_field_using_string_arguments (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
--- FAIL: RelationTest#test_size_with_eager_loading_and_custom_order_and_distinct (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_run_add_callback_methods_for_has_many (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_saving_multiple_relationships (unexpected)
--- FAIL: RelationTest#test_joins_with_string_array (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_removing_associations_on_destroy (unexpected)
--- FAIL: AssociationsJoinModelTest#test_include_polymorphic_has_one (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_run_remove_callback_procs_for_has_many (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_adding_type_mismatch (unexpected)
--- FAIL: RelationTest#test_reorder_with_first (unexpected)
--- FAIL: FinderTest#test_find_by_ids_missing_one (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
--- FAIL: RelationTest#test_finding_with_reversed_assoc_order (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through_with_nonstandard_id (unexpected)
--- FAIL: DefaultScopingTest#test_with_abstract_class_where_clause_should_not_be_duplicated (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_and_one_level (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_preloading_across_has_one_constrains_loaded_records (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9373184?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9373184?buildTab=artifacts#/activerecord) on master @ [99102ddf4b7602788b422366f1acc14b81c64d24](https://github.com/cockroachdb/cockroach/commits/99102ddf4b7602788b422366f1acc14b81c64d24):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-99102ddf4b7602788b422366f1acc14b81c64d24
Tests run against activerecord 7.0.3
7163 Total Tests Run
7127 tests passed
36 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_version_contains_key_and_version_regardless_of_collection_cache_versioning_setting (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9373173?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9373173?buildTab=artifacts#/activerecord) on master @ [99102ddf4b7602788b422366f1acc14b81c64d24](https://github.com/cockroachdb/cockroach/commits/99102ddf4b7602788b422366f1acc14b81c64d24):


```
--- FAIL: FinderTest#test_model_class_responds_to_second_bang (unexpected)
--- FAIL: ActiveRecord::AdapterPreventWritesTest#test_errors_when_an_insert_query_prefixed_by_a_slash_star_comment_containing_read_command_is_called_while_preventing_writes (unexpected)
--- FAIL: BelongsToAssociationsTest#test_natural_assignment (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_replace_target_record (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_many_with_has_and_belongs_to_many_source_reflection_preload_via_joins (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_value_is_properly_quoted (unexpected)
--- FAIL: FinderTest#test_third_with_offset (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_column (unexpected)
--- FAIL: FinderTest#test_exists_with_distinct_and_offset_and_joins (unexpected)
--- FAIL: FinderTest#test_first_have_determined_order_by_default (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_one_mapping_and_key_value_being_attribute_value (unexpected)
--- FAIL: AssociationsTest#test_loading_the_association_target_should_keep_child_records_marked_for_destruction (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionHandlersMultiPoolConfigTest#test_establish_connection_with_pool_configs (unexpected)
--- FAIL: HasOneThroughDisableJoinsAssociationsTest#test_nil_on_disable_joins_through (unexpected)
--- FAIL: BelongsToAssociationsTest#test_natural_assignment_to_nil_with_primary_key (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_range (unexpected)
--- FAIL: ActiveRecord::Migration::IndexTest#test_add_index_which_already_exists_does_not_raise_error_with_option (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_many_raises_exception (unexpected)
--- FAIL: AutomaticInverseFindingTests#test_has_one_and_belongs_to_with_non_default_foreign_key_should_not_find_inverse_automatically (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionPoolTest#test_removing_releases_latch (unexpected)
--- FAIL: NumericalityValidationTest#test_column_with_precision_higher_than_double_fig (unexpected)
--- FAIL: AutomaticInverseFindingTests#test_has_one_and_belongs_to_with_a_scope_and_automatic_scope_inversing_should_find_inverse_automatically (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_association_create_constructor_creates_through_record (unexpected)
--- FAIL: EnumTest#test_validate_inclusion_of_value_in_array (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_be_possible_to_destroy_a_record (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_polymorphic_changes (unexpected)
--- FAIL: TimestampTest#test_touching_many_attributes_updates_them (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnsTest#test_rename_column_with_sql_reserved_word (unexpected)
--- FAIL: BelongsToAssociationsTest#test_dependency_should_halt_parent_destruction (unexpected)
--- FAIL: FinderTest#test_find_with_ids_and_order_clause (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_an_invalid_type (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_and_reassigned_parent (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_conditions_on_source_associations_preload_via_joins (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionHandlersShardingDbTest#test_same_shards_across_clusters (unexpected)
--- FAIL: ActiveRecord::OrTest#test_or_inside_named_scope (unexpected)
--- FAIL: ActiveRecord::Encryption::MessageSerializerTest#test_raises_Decryption_when_trying_to_parse_message_with_more_than_one_nested_message (unexpected)
--- FAIL: PostgresqlEnumTest#test_column (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_conditions_on_through_associations (unexpected)
--- FAIL: AutomaticInverseFindingTests#test_polymorphic_has_one_should_find_inverse_automatically (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_counter_cache (unexpected)
--- FAIL: QueryLogsTest#test_custom_proc_tags (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_polymorphic_disable_joins_through_reordering (unexpected)
--- FAIL: BelongsToAssociationsTest#test_create_with_block (unexpected)
--- FAIL: FinderTest#test_named_bind_variables (unexpected)
--- FAIL: NumericalityValidationTest#test_column_with_precision (unexpected)
--- FAIL: DDLTest#test_create_simple_geometry (unexpected)
--- FAIL: CallbacksOnDestroyUpdateActionRaceTest#test_trigger_on_update_where_row_was_deleted (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9382869?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9382869?buildTab=artifacts#/activerecord) on master @ [1e27c65fb5a3810a5e2285cf07b45d95d401cca6](https://github.com/cockroachdb/cockroach/commits/1e27c65fb5a3810a5e2285cf07b45d95d401cca6):


```
--- FAIL: HasManyAssociationsTest#test_dont_call_save_callbacks_twice_on_has_many (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_incorrectly_ordered_through_associations (unexpected)
--- FAIL: HasManyAssociationsTest#test_replace_with_less (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_executes_limited_query (unexpected)
--- FAIL: SignedIdTest#test_find_signed_record (unexpected)
--- FAIL: FinderTest#test_last_bang_present (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_copies_arel_bind_params (unexpected)
--- FAIL: BasicsTest#test_limit_without_comma (unexpected)
--- FAIL: PrimaryKeysTest#test_update_counters_should_quote_pkey_and_quote_counter_columns (unexpected)
--- FAIL: BasicsTest#test_attributes_on_dummy_time (unexpected)
--- FAIL: CalculationsTest#test_should_group_by_summed_association (unexpected)
--- FAIL: BasicsTest#test_#connecting_to_with_role_and_shard (unexpected)
--- FAIL: FinderTest#test_exists_with_aggregate_having_three_mappings_with_one_difference (unexpected)
--- FAIL: HasManyAssociationsTest#test_default_scope_on_relations_is_not_cached (unexpected)
--- FAIL: UpdateAllTest#test_update_all_with_order_and_limit_and_offset_updates_subset_only (unexpected)
--- FAIL: RelationTest#test_order_by_relation_attribute (unexpected)
--- FAIL: HasManyAssociationsTest#test_anonymous_has_many (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_collection_singular_ids_getter_with_string_primary_keys (unexpected)
--- FAIL: TimestampTest#test_touching_an_attribute_updates_timestamp_with_given_time (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: RelationTest#test_optimizer_hints_with_blank_value (unexpected)
--- FAIL: FinderTest#test_exists_with_polymorphic_relation (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_creating_association_builds_through_record_for_new (unexpected)
--- FAIL: BasicsTest#test_dup_of_saved_object_marks_as_dirty_only_changed_attributes (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_has_many_association_through_a_belongs_to_association_where_the_association_doesnt_exist (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_new_when_explicitly_passing_value (unexpected)
--- FAIL: RelationTest#test_reverse_order_with_function (unexpected)
--- FAIL: CalculationsTest#test_pluck_with_empty_in (unexpected)
--- FAIL: HasManyAssociationsTest#test_update_all_respects_association_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_the_association_with_an_array (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_one_mapping_and_key_value_being_attribute_value (unexpected)
--- FAIL: InverseBelongsToTests#test_building_has_many_parent_association_inverses_one_record (unexpected)
--- FAIL: FinderTest#test_find_by_on_attribute_that_is_a_reserved_word (unexpected)
--- FAIL: UpdateAllTest#test_update_counters_cares_about_optimistic_locking (unexpected)
--- FAIL: RelationTest#test_find_all_using_where_twice_should_or_the_relation (unexpected)
--- FAIL: CalculationsTest#test_from_option_with_table_different_than_class (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_repeating (unexpected)
--- FAIL: CalculationsTest#test_should_group_by_summed_field_with_conditions (unexpected)
--- FAIL: CalculationsTest#test_should_group_by_scoped_field (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_delete_through_belongs_to_with_dependent_nullify (unexpected)
--- FAIL: PersistenceTest#test_class_level_update_without_ids (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: BasicsTest#test_clear_cache_when_setting_table_name (unexpected)
--- FAIL: SignedIdTest#test_use_a_custom_verifier (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_with_scope_that_has_joined_same_table_with_parent_relation (unexpected)
--- FAIL: UpdateAllTest#test_update_all_ignores_order_without_limit_from_association (unexpected)
--- FAIL: InverseHasManyTests#test_parent_instance_should_be_shared_with_eager_loaded_children (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9417448?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9417448?buildTab=artifacts#/activerecord) on master @ [07c7d4b3abfa1dc79d8fda385132a27c81665fee](https://github.com/cockroachdb/cockroach/commits/07c7d4b3abfa1dc79d8fda385132a27c81665fee):


```
--- FAIL: FoxyFixturesTest#test_strips_DEFAULTS_key (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_recursive_cascading_four_levels_has_many_through (unexpected)
--- FAIL: FinderTest#test_second_have_primary_key_order_by_default (unexpected)
--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_strict_loading_belongs_to_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: CounterCacheTest#test_update_other_counters_on_parent_destroy (unexpected)
--- FAIL: PrimaryKeysTest#test_primary_key_returns_value_if_it_exists (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_calls_updated_at_when_the_value_is_generated_at_create_time (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_has_many_sti_and_subclasses (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ReferencesForeignKeyTest#test_foreign_keys_can_be_created_while_changing_the_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ReferencesForeignKeyTest#test_foreign_key_methods_respect_pluralize_table_names (unexpected)
--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_timestamp (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_and_self_table_reference (unexpected)
--- FAIL: FinderTest#test_find_by_two_attributes_that_are_both_aggregates (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_last (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: FoxyFixturesTest#test_supports_inline_habtm (unexpected)
--- FAIL: FinderTest#test_unexisting_record_exception_handling (unexpected)
--- FAIL: FinderTest#test_find_some_message_with_custom_primary_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
--- FAIL: FinderTest#test_find_by_association_subquery (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_belongs_to_relation_if_strict_loading_by_default (unexpected)
--- FAIL: RelationScopingTest#test_scoping_with_query_method_works_in_the_scope_block (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_strict_loading_has_one_relation (unexpected)
--- FAIL: FoxyFixturesTest#test_supports_sti_with_respective_files (unexpected)
--- FAIL: ActiveRecord::Encryption::EncryptableRecordTest#test_encrypts_serialized_attributes (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_add_column_with_precision_and_scale (unexpected)
--- FAIL: FinderTest#test_find_by_with_multi-arg_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: CounterCacheTest#test_the_passed_symbol_needs_to_be_an_association_name_or_counter_name (unexpected)
--- FAIL: FinderTest#test_last_bang_present (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_select (unexpected)
--- FAIL: RelationScopingTest#test_reverse_order (unexpected)
--- FAIL: RelationScopingTest#test_scoped_unscoped (unexpected)
--- FAIL: FinderTest#test_joins_dont_clobber_id (unexpected)
--- FAIL: HotCompatibilityTest#test_insert_after_remove_column (unexpected)
--- FAIL: DelegatedTypeTest#test_association_uuid (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: FinderTest#test_#skip_query_cache!_for_#exists? (unexpected)
--- FAIL: FinderTest#test_take_failing (unexpected)
--- FAIL: PrimaryKeysTest#test_integer_key (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_true (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ColumnsTest#test_change_column_default_with_from_and_to (unexpected)
--- FAIL: RelationScopingTest#test_reverse_order_with_arel_node (unexpected)
--- FAIL: FinderTest#test_fourth_have_primary_key_order_by_default (unexpected)
--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_strict_loading_habtm_relation (unexpected)
--- FAIL: FoxyFixturesTest#test_automatically_sets_primary_key (unexpected)
--- FAIL: FinderTest#test_exists_with_distinct_association_includes_and_limit (unexpected)
--- FAIL: FinderTest#test_include_on_unloaded_relation_with_having_referencing_aliased_select (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9482206?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9482206?buildTab=artifacts#/activerecord) on master @ [8e6f530644eb35313a479af529b1f672c12dca38](https://github.com/cockroachdb/cockroach/commits/8e6f530644eb35313a479af529b1f672c12dca38):


```
--- FAIL: RelationTest#test_named_extension (unexpected)
--- FAIL: RelationTest#test_select_with_blank_value (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_on_has_many (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_bang_with_block (unexpected)
--- FAIL: HasOneAssociationsTest#test_assignment_before_child_saved (unexpected)
--- FAIL: RelationTest#test_find_or_initialize_by (unexpected)
--- FAIL: RelationTest#test_create_bang_with_array (unexpected)
--- FAIL: DDLTest#test_create_simple_geometry_using_shortcut (unexpected)
--- FAIL: AssociationProxyTest#test_pluck_uses_loaded_target (unexpected)
--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_datetime_doesnt_set_precision_on_change_table_5_1 (unexpected)
--- FAIL: RelationTest#test_#where_with_empty_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
--- FAIL: RelationTest#test_default_reverse_order_on_table_without_primary_key (unexpected)
--- FAIL: RelationTest#test_where_relation_with_delegated_ar_object (unexpected)
--- FAIL: RelationTest#test_group_with_subquery_string_in_from_does_not_use_original_table_name (unexpected)
--- FAIL: RelationTest#test_count_explicit_columns (unexpected)
--- FAIL: RelationTest#test_reverse_order_with_function_other_predicates (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_association_with_bang_failing (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_with_block (unexpected)
--- FAIL: RelationTest#test_annotate_with_blank_value (unexpected)
--- FAIL: RelationTest#test_finding_with_desc_order_with_string (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyTest#test_schema_dumping_with_custom_fk_ignore_pattern (unexpected)
--- FAIL: HasOneAssociationsTest#test_type_mismatch (unexpected)
--- FAIL: RelationTest#test_create_with_nested_attributes (unexpected)
--- FAIL: DDLTest#test_null_constraints (unexpected)
--- FAIL: RelationTest#test_loaded_relations_cannot_be_mutated_by_extending! (unexpected)
--- FAIL: RelationTest#test_find_by!_raises_RecordNotFound_if_the_record_is_missing (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyInCreateTest#test_foreign_keys_can_be_created_with_the_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
--- FAIL: RelationTest#test_find_with_included_associations (unexpected)
--- FAIL: PostgresqlIntervalTest#test_interval_type_cast_string_and_numeric_from_user (unexpected)
--- FAIL: RelationTest#test_select_with_subquery_in_from_uses_original_table_name (unexpected)
--- FAIL: AssociationProxyTest#test_push_followed_by_save_does_not_load_target (unexpected)
--- FAIL: ReservedWordTest#test_associations_work_with_reserved_words (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnAHasManyAssociationWithAcceptsNestedAttributes#test_errors_details_should_be_indexed_when_passed_as_array (unexpected)
--- FAIL: RelationTest#test_automatically_added_reorder_references (unexpected)
--- FAIL: RelationTest#test_intersection_with_array (unexpected)
--- FAIL: AssociationProxyTest#test_load_does_load_target (unexpected)
--- FAIL: AssociationProxyTest#test_proxy_object_can_be_stubbed (unexpected)
--- FAIL: RelationTest#test_finding_with_subquery_with_eager_loading_in_where (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
--- FAIL: RelationTest#test_scoped_first (unexpected)
--- FAIL: PostgresqlBitStringTest#test_bit_string_column (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_before_save (unexpected)
--- FAIL: RelationTest#test_default_scope_order_with_scope_order (unexpected)
--- FAIL: RelationTest#test_create_or_find_by_with_bang_within_transaction (unexpected)
--- FAIL: ReservedWordTest#test_has_and_belongs_to_many (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_with_scopes (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9498741?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9498741?buildTab=artifacts#/activerecord) on master @ [85e41ca8d3d9edacf5ee3061a2591b159b9b0502](https://github.com/cockroachdb/cockroach/commits/85e41ca8d3d9edacf5ee3061a2591b159b9b0502):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-85e41ca8d3d9edacf5ee3061a2591b159b9b0502
Tests run against activerecord 7.0.3
7163 Total Tests Run
7126 tests passed
37 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
29 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::Encryption::EncryptionPerformanceTest#test_performance_when_saving_records (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_version_contains_key_and_version_regardless_of_collection_cache_versioning_setting (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9507168?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9507168?buildTab=artifacts#/activerecord) on master @ [85e41ca8d3d9edacf5ee3061a2591b159b9b0502](https://github.com/cockroachdb/cockroach/commits/85e41ca8d3d9edacf5ee3061a2591b159b9b0502):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-85e41ca8d3d9edacf5ee3061a2591b159b9b0502
Tests run against activerecord 7.0.3
7163 Total Tests Run
7127 tests passed
36 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_version_contains_key_and_version_regardless_of_collection_cache_versioning_setting (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9516059?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9516059?buildTab=artifacts#/activerecord) on master @ [85e41ca8d3d9edacf5ee3061a2591b159b9b0502](https://github.com/cockroachdb/cockroach/commits/85e41ca8d3d9edacf5ee3061a2591b159b9b0502):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-85e41ca8d3d9edacf5ee3061a2591b159b9b0502
Tests run against activerecord 7.0.3
7163 Total Tests Run
7127 tests passed
36 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_version_contains_key_and_version_regardless_of_collection_cache_versioning_setting (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9538080?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9538080?buildTab=artifacts#/activerecord) on master @ [a5d61d2cf2bfc11bd6bdc27dcab32467334fb123](https://github.com/cockroachdb/cockroach/commits/a5d61d2cf2bfc11bd6bdc27dcab32467334fb123):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-a5d61d2cf2bfc11bd6bdc27dcab32467334fb123
Tests run against activerecord 7.0.3
7163 Total Tests Run
7127 tests passed
36 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_version_contains_key_and_version_regardless_of_collection_cache_versioning_setting (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9554120?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9554120?buildTab=artifacts#/activerecord) on master @ [582880ae2bb5d08e83a9871735deeeb3ed892e7d](https://github.com/cockroachdb/cockroach/commits/582880ae2bb5d08e83a9871735deeeb3ed892e7d):


```
--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_should_ignore_new_associated_records_with_truthy_destroy_attribute (unexpected)
--- FAIL: FoxyFixturesTest#test_namespaced_models (unexpected)
--- FAIL: FixturesTest#test_binary_in_fixtures (unexpected)
--- FAIL: HasManyAssociationsTest#test_building_the_associated_object_with_explicit_sti_base_class (unexpected)
--- FAIL: RelationTest#test_size_with_eager_loading_and_manual_distinct_select_and_custom_order (unexpected)
--- FAIL: AssociationsJoinModelTest#test_deleting_junk_from_has_many_through_should_raise_type_mismatch (unexpected)
--- FAIL: BasicsTest#test_#connecting_to_with_role (unexpected)
--- FAIL: DefaultScopingTest#test_default_scope_with_all_queries_runs_on_select (unexpected)
--- FAIL: FinderTest#test_find_on_association_proxy_conditions (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_empty_on_an_association_that_has_been_loaded_does_not_performs_query (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_where_with_emoji_for_binary_column (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_associate_with_create_bang_and_invalid_options (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_doesn't_touch_timestamps_with_touch:_[] (unexpected)
--- FAIL: HasManyAssociationsTest#test_depends_and_nullify_on_polymorphic_assoc (unexpected)
--- FAIL: ActiveRecord::WhereTest#test_belongs_to_array_value_where (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: RelationTest#test_order_triggers_eager_loading_when_ordering_using_symbols (unexpected)
--- FAIL: RelationTest#test_find_or_initialize_by (unexpected)
--- FAIL: CockroachDB::EnumTest#test_enum_logs_a_warning_if_auto-generated_negative_scopes_would_clash_with_other_enum_names (unexpected)
--- FAIL: FixturesTest#test_fixtures_are_set_up_with_database_env_variable (unexpected)
--- FAIL: HasManyAssociationsTest#test_association_size_calculation_works_with_default_scoped_selects_when_not_previously_fetched (unexpected)
--- FAIL: TransactionCallbacksTest#test_only_call_after_commit_on_create_and_doesnt_leaky (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionHandlersMultiDbTest#test_connects_to_using_top_level_key_in_two_level_config (unexpected)
--- FAIL: TouchLaterTest#test_touch_later_respects_no_touching_policy (unexpected)
--- FAIL: RelationTest#test_joins_with_select (unexpected)
--- FAIL: ModulesTest#test_module_spanning_has_and_belongs_to_many_associations (unexpected)
--- FAIL: HasManyAssociationsTest#test_counter_cache_updates_in_memory_after_update_with_inverse_of_enabled (unexpected)
--- FAIL: DirtyTest#test_reverted_changes_are_not_dirty_going_from_nil_to_value_and_back (unexpected)
--- FAIL: TestNestedAttributesOnAHasOneAssociation#test_should_work_with_update_as_well (unexpected)
--- FAIL: HasManyAssociationsTest#test_get_ids_for_loaded_associations (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: NestedAttributesWithCallbacksTest#test_:before_add_called_for_new_bird_when_not_loaded (unexpected)
--- FAIL: RelationTest#test_select_with_from_includes_original_table_name (unexpected)
--- FAIL: SignedIdTest#test_find_signed_record_with_a_bang_for_single_table_inheritance_(STI_Models) (unexpected)
--- FAIL: IntegrationTest#test_to_param_returns_id_if_not_persisted_but_id_is_set (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_all_on_desynced_counter_cache_association (unexpected)
--- FAIL: RelationTest#test_find_by_doesn't_have_implicit_ordering (unexpected)
--- FAIL: BasicsTest#test_dup_of_saved_object_marks_as_dirty_only_changed_attributes (unexpected)
--- FAIL: TransactionCallbacksTest#test_only_call_after_commit_on_destroy_after_transaction_commits_for_destroyed_record (unexpected)
--- FAIL: RelationTest#test_no_arguments_to_joins_raise_errors (unexpected)
--- FAIL: HasManyAssociationsTest#test_new_the_association_with_an_array (unexpected)
--- FAIL: BasicsTest#test_set_table_name_symbol_converted_to_string (unexpected)
--- FAIL: BelongsToAssociationsTest#test_dependency_should_halt_parent_destruction_with_cascaded_three_levels (unexpected)
--- FAIL: FinderTest#test_find_with_proc_parameter_and_block (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_polymorphic_with_rewhere (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_distinct_option_prevents_duplicate_push (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9577967?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9577967?buildTab=artifacts#/activerecord) on master @ [678cfd4cbebbf3cced16747baf42a4c54cb2a92d](https://github.com/cockroachdb/cockroach/commits/678cfd4cbebbf3cced16747baf42a4c54cb2a92d):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-678cfd4cbebbf3cced16747baf42a4c54cb2a92d
Tests run against activerecord 7.0.3
7163 Total Tests Run
7127 tests passed
36 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_version_contains_key_and_version_regardless_of_collection_cache_versioning_setting (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0 release-blocker]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9599062?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestFipsNightlyGceBazel/9599062?buildTab=artifacts#/activerecord) on master @ [8e6cdbd8b3d12ba5d65db101a3abfeb2cb1bf470](https://github.com/cockroachdb/cockroach/commits/8e6cdbd8b3d12ba5d65db101a3abfeb2cb1bf470):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-8e6cdbd8b3d12ba5d65db101a3abfeb2cb1bf470
Tests run against activerecord 7.0.3
7163 Total Tests Run
7127 tests passed
36 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
28 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_collection_cache_versioning_is_enabled (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_version_contains_key_and_version_regardless_of_collection_cache_versioning_setting (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_limit (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_empty_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_relation_cache_key_changes_when_the_sql_query_changes (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_cache_key_on_model (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0 release-blocker]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9627042?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9627042?buildTab=artifacts#/activerecord) on master @ [c08946af1f935d543a033de6b7b585c4d65288c4](https://github.com/cockroachdb/cockroach/commits/c08946af1f935d543a033de6b7b585c4d65288c4):


```
--- FAIL: RelationTest#test_delete_by (unexpected)
--- FAIL: PostGISTest#test_set_point_bad_wkt (unexpected)
--- FAIL: RelationTest#test_pluck_with_from_includes_original_table_name (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert_change_column_comment (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_restrict_foreign_key (unexpected)
--- FAIL: SerializedAttributeTest#test_newly_emptied_serialized_hash_is_changed (unexpected)
--- FAIL: RelationTest#test_loaded_first_with_limit (unexpected)
--- FAIL: RelationTest#test_find_by_id_with_list_of_ar (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
--- FAIL: RelationTest#test_preload_applies_to_all_chained_preloaded_scopes (unexpected)
--- FAIL: RelationTest#test_optimizer_hints_with_blank_value (unexpected)
--- FAIL: RelationTest#test_size_with_eager_loading_and_custom_order_and_distinct (unexpected)
--- FAIL: RelationTest#test_find_all_using_where_with_relation_and_alternate_primary_key (unexpected)
--- FAIL: RelationTest#test_presence (unexpected)
--- FAIL: RelationTest#test_find_all_using_where_with_relation_with_joins (unexpected)
--- FAIL: RelationTest#test_pluck_with_subquery_in_from_uses_original_table_name (unexpected)
--- FAIL: ReflectionTest#test_scope_chain_of_polymorphic_association_does_not_leak_into_other_hmt_associations (unexpected)
--- FAIL: DelegatedTypeTest#test_delegated_type_predicates (unexpected)
--- FAIL: RelationTest#test_many (unexpected)
--- FAIL: ReflectionTest#test_association_reflection_in_modules (unexpected)
--- FAIL: RelationTest#test_locked_should_not_build_arel (unexpected)
--- FAIL: RelationTest#test_create_or_find_by_with_bang_should_raise_due_to_validation_errors (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert_add_index_with_name (unexpected)
--- FAIL: RelationTest#test_support_upper_and_lower_case_directions (unexpected)
--- FAIL: RelationTest#test_find_by!_doesn't_have_implicit_ordering (unexpected)
--- FAIL: RelationTest#test_automatically_added_where_not_references (unexpected)
--- FAIL: RelationTest#test_no_arguments_to_references_raise_errors (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_legacy_up (unexpected)
--- FAIL: RelationTest#test_do_not_double_quote_string_id (unexpected)
--- FAIL: RelationTest#test_unscope_grouped_where (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert (unexpected)
--- FAIL: RelationTest#test_new_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: RelationTest#test_count_with_distinct (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_on_custom_attribute_with_default (unexpected)
--- FAIL: RelationTest#test_create_or_find_by_with_bang (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
--- FAIL: ModulesTest#test_module_table_name_suffix_with_global_suffix (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_boolean_value_false (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
--- FAIL: RelationTest#test_select_with_aggregates (unexpected)
--- FAIL: ModulesTest#test_module_table_name_suffix (unexpected)
--- FAIL: CockroachDB::PostgresqlBigSerialTest#test_bigserial_column (unexpected)
--- FAIL: RelationTest#test_preloading_with_associations_and_merges (unexpected)
--- FAIL: RelationTest#test_typecasting_where_with_array (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_up (unexpected)
--- FAIL: RelationTest#test_find_by!_with_hash_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: RelationTest#test_find_or_create_by! (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9749895?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9749895?buildTab=artifacts#/activerecord) on master @ [23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b](https://github.com/cockroachdb/cockroach/commits/23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-23ea65e4b5cfaa5e02298cddfbe8f9fbe581456b
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_suffix (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_prefix (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9781989?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9781989?buildTab=artifacts#/activerecord) on master @ [847de4595b62a0c03b14a3d0865cc8222fcf0f61](https://github.com/cockroachdb/cockroach/commits/847de4595b62a0c03b14a3d0865cc8222fcf0f61):


```
--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_collection_size_from_params (unexpected)
--- FAIL: OptimisticLockingTest#test_update_with_dirty_primary_key (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_strict_loading_has_one_relation (unexpected)
--- FAIL: HasManyAssociationsTest#test_counting_with_association_limit (unexpected)
--- FAIL: FoxyFixturesTest#test_automatically_sets_primary_key (unexpected)
--- FAIL: FinderTest#test_find_by_on_a_scope_does_not_perform_statement_caching (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_automatically_save_the_associated_models (unexpected)
--- FAIL: EagerAssociationTest#test_eager_load_multiple_associations_with_references (unexpected)
--- FAIL: HasManyAssociationsTest#test_counting_with_single_hash (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_an_sti_join_model (unexpected)
--- FAIL: HasManyAssociationsTest#test_creation_respects_hash_condition (unexpected)
--- FAIL: BelongsToAssociationsTest#test_create_bang_with_conditions (unexpected)
--- FAIL: FinderTest#test_third_have_primary_key_order_by_default (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_updates_foreign_id_field_for_new_and_saved_records (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_foreign_keys (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_all (unexpected)
--- FAIL: HasManyAssociationsTest#test_dependence_with_transaction_support_on_failure (unexpected)
--- FAIL: RelationScopingTest#test_circular_joins_with_scoping_does_not_crash (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_polymorphic_has_one (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_updating_attributes_on_non_rich_associations (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_first_after_reset (unexpected)
--- FAIL: DelegatedTypeTest#test_association_uuid (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_non_standard_table_names (unexpected)
--- FAIL: HasManyAssociationsTest#test_replace (unexpected)
--- FAIL: DDLTest#test_drop_geography_column (unexpected)
--- FAIL: FinderTest#test_exists_with_eager_load (unexpected)
--- FAIL: UpdateAllTest#test_update_counters_with_joins (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_by_integer_id (unexpected)
--- FAIL: BelongsToAssociationsTest#test_find_by_with_custom_primary_key (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type_cast_from_invalid_string (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_increment_counter_updates_custom_lock_version (unexpected)
--- FAIL: EagerAssociationTest#test_preloading_of_instance_dependent_associations_is_supported (unexpected)
--- FAIL: EagerAssociationTest#test_eager_load_belongs_to_quotes_table_and_column_names (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_include (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_going_through_polymorphic_join_model_with_custom_primary_key (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_1#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: OptimisticLockingTest#test_yaml_dumping_with_lock_column (unexpected)
--- FAIL: AssociationsJoinModelTest#test_eager_load_has_many_through_has_many_with_conditions (unexpected)
--- FAIL: FilterAttributesTest#test_filter_attributes_should_handle_[FILTERED]_value_properly (unexpected)
--- FAIL: ActiveRecord::DatabaseTasksTruncateAllTest#test_truncate_tables (unexpected)
--- FAIL: BelongsToAssociationsTest#test_dont_find_target_when_saving_foreign_key_after_stale_association_loaded (unexpected)
--- FAIL: EagerAssociationTest#test_eager-loading_readonly_association (unexpected)
--- FAIL: DDLTest#test_column_defaults (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_include_returns_false_for_non_matching_record_to_verify_scoping (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_second_to_last_bang (unexpected)
--- FAIL: FoxyFixturesTest#test_supports_sti (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9848398?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9848398?buildTab=artifacts#/activerecord) on master @ [3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f](https://github.com/cockroachdb/cockroach/commits/3b1bc872bb4bf8e5eb2ad8790ba8ec781167dd1f):


```
--- FAIL: InheritanceTest#test_alt_eager_loading (unexpected)
--- FAIL: InheritanceTest#test_inheritance_find_all (unexpected)
--- FAIL: InheritanceTest#test_should_store_demodulized_class_name_with_store_full_sti_class_option_disabled (unexpected)
--- FAIL: InheritanceTest#test_new_with_ar_base (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column (unexpected)
--- FAIL: InheritanceTest#test_eager_load_belongs_to_primary_key_quoting (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_not_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_infers_column (unexpected)
--- FAIL: InheritanceTest#test_new_with_autoload_paths (unexpected)
--- FAIL: InheritanceTest#test_inherits_custom_primary_key (unexpected)
--- FAIL: InheritanceTest#test_destroy_all_within_inheritance (unexpected)
--- FAIL: InheritanceTest#test_new_with_complex_inheritance (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_constraint_by_name (unexpected)
--- FAIL: InheritanceTest#test_compute_type_argument_error (unexpected)
--- FAIL: InheritanceTest#test_becomes_and_change_tracking_for_inheritance_columns (unexpected)
--- FAIL: InheritanceTest#test_new_with_invalid_type (unexpected)
--- FAIL: InheritanceTest#test_where_create_bang_with_unrelated_type (unexpected)
--- FAIL: PostgresqlNumberTest#test_data_type (unexpected)
--- FAIL: InheritanceTest#test_abstract_class (unexpected)
--- FAIL: InheritanceTest#test_alt_update_all_within_inheritance (unexpected)
--- FAIL: InheritanceTest#test_company_descends_from_active_record (unexpected)
--- FAIL: InheritanceTest#test_complex_inheritance (unexpected)
--- FAIL: InheritanceTest#test_descends_from_active_record (unexpected)
--- FAIL: InheritanceTest#test_becomes_sets_variables_before_initialization_callbacks (unexpected)
--- FAIL: InheritanceTest#test_compute_type_success (unexpected)
--- FAIL: InheritanceTest#test_find_first_within_inheritance (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column_with_full_inserts (unexpected)
--- FAIL: InheritanceTest#test_inheritance_without_mapping (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
--- FAIL: PostgresqlNumberTest#test_update (unexpected)
--- FAIL: InheritanceTest#test_abstract_inheritance_base_class (unexpected)
--- FAIL: InheritanceTest#test_inheritance_new_with_default_class (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
--- FAIL: InheritanceTest#test_alt_inheritance_save (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
--- FAIL: InheritanceTest#test_should_store_full_class_name_with_store_full_sti_class_option_enabled (unexpected)
--- FAIL: InheritanceTest#test_alt_inheritance_find_all (unexpected)
--- FAIL: InheritanceTest#test_inheritance_save (unexpected)
--- FAIL: InheritanceTest#test_class_with_store_full_sti_class_returns_full_name (unexpected)
--- FAIL: InheritanceTest#test_alt_inheritance_find (unexpected)
--- FAIL: InheritanceTest#test_where_create_with_subclass (unexpected)
--- FAIL: InheritanceTest#test_new_with_unrelated_namespaced_type (unexpected)
--- FAIL: InheritanceTest#test_where_new_with_unrelated_type (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
--- FAIL: InheritanceTest#test_where_new_with_subclass (unexpected)
--- FAIL: InheritanceTest#test_alt_finding_incorrect_type_data (unexpected)
--- FAIL: InheritanceTest#test_base_class_activerecord_error (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9880074?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9880074?buildTab=artifacts#/activerecord) on master @ [7d02e061759599a2b054ee06c71524443f77d20a](https://github.com/cockroachdb/cockroach/commits/7d02e061759599a2b054ee06c71524443f77d20a):


```
--- FAIL: HasManyAssociationsTest#test_modifying_a_through_a_has_many_should_raise (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_belongs_to (unexpected)
--- FAIL: BelongsToAssociationsTest#test_assigning_an_association_doesn't_result_in_duplicate_objects (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all (unexpected)
--- FAIL: BelongsToAssociationsTest#test_should_set_foreign_key_on_create_association (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
--- FAIL: HasManyAssociationsTest#test_attributes_are_set_when_initialized_from_has_many_null_relationship (unexpected)
--- FAIL: ReadOnlyTest#test_has_many_find_readonly (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_many_through_has_many (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_column (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_with_bang_on_habtm_when_parent_is_new_raises (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_via_block (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroying (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_reassign_with_namespaced_models_and_counters (unexpected)
--- FAIL: HasManyAssociationsTest#test_collection_proxy_respects_default_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_dependent_when_deleted_from_association (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_foreign_type_field_updating (unexpected)
--- FAIL: StrictLoadingTest#test_preload_audit_logs_are_strict_loading_because_parent_is_strict_loading (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_on_multiple_records (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_singleton_has_many_through (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_without_loading_association (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
--- FAIL: AssociationsTest#test_using_limitable_reflections_helper (unexpected)
--- FAIL: HasManyAssociationsTest#test_exists_respects_association_scope (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_with_dirty_target (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading! (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_not_set (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_without_counter_cache_option (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_from_association_sets_inverse_instance (unexpected)
--- FAIL: StrictLoadingTest#test_eager_load_audit_logs_are_strict_loading_because_it_is_strict_loading_by_default (unexpected)
--- FAIL: HasManyAssociationsTest#test_counting_with_association_limit (unexpected)
--- FAIL: HasManyAssociationsTest#test_associations_autosaves_when_object_is_already_persisted (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_inferes_column (unexpected)
--- FAIL: StrictLoadingTest#test_raises_if_strict_loading_by_default_and_lazy_loading (unexpected)
--- FAIL: HasManyAssociationsTest#test_include_checks_if_record_exists_if_target_not_loaded (unexpected)
--- FAIL: HasManyAssociationsTest#test_counting (unexpected)
--- FAIL: ReadOnlyTest#test_cant_save_readonly_record (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_defaults_are_not_touched_on_the_columns (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter_with_reassigning (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_restrict_foreign_key (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_touch_with_reassigning (unexpected)
--- FAIL: HasManyAssociationsTest#test_ids_reader_cache_should_be_cleared_when_collection_is_deleted (unexpected)
--- FAIL: HasManyAssociationsTest#test_dynamic_find_should_respect_association_order_for_through (unexpected)
--- FAIL: HasManyAssociationsTest#test_associations_order_should_be_priority_over_throughs_order (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_by_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9904268?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/9904268?buildTab=artifacts#/activerecord) on master @ [ed08739c22ecd246c9745d0f301976023c0d004b](https://github.com/cockroachdb/cockroach/commits/ed08739c22ecd246c9745d0f301976023c0d004b):


```
--- FAIL: PersistenceTest#test_update_many_with_duplicated_ids (unexpected)
--- FAIL: InsertAllTest#test_insert_all_and_upsert_all_works_with_composite_primary_keys_when_unique_by_is_provided (unexpected)
--- FAIL: BasicsTest#test_slice (unexpected)
--- FAIL: BasicsTest#test_default_in_utc (unexpected)
--- FAIL: BasicsTest#test_auto_id (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
--- FAIL: BasicsTest#test_ignored_columns_are_stored_as_an_array_of_string (unexpected)
--- FAIL: BasicsTest#test_marshal_between_processes (unexpected)
--- FAIL: BasicsTest#test_can_call_connected_to_with_role_and_shard_on_abstract_classes (unexpected)
--- FAIL: OptimisticLockingTest#test_removing_has_and_belongs_to_many_associations_upon_destroy (unexpected)
--- FAIL: BasicsTest#test_group_weirds_by_from (unexpected)
--- FAIL: PersistenceTest#test_update_columns_should_not_modify_updated_at (unexpected)
--- FAIL: BasicsTest#test_create_after_initialize_with_array_param (unexpected)
--- FAIL: BasicsTest#test_marshalling_with_associations (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_name (unexpected)
--- FAIL: BasicsTest#test_#connecting_to_with_role_and_shard (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_new_when_explicitly_passing_value (unexpected)
--- FAIL: DefaultScopingTest#test_with_abstract_class_where_clause_should_not_be_duplicated (unexpected)
--- FAIL: BasicsTest#test_unicode_column_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: BasicsTest#test_previously_changed (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column (unexpected)
--- FAIL: BasicsTest#test_limit_should_sanitize_sql_injection_for_limit_with_commas (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_no_pk_join_table_append (unexpected)
--- FAIL: AssociationsExtensionsTest#test_named_extension_on_habtm (unexpected)
--- FAIL: PersistenceTest#test_increment_updates_counter_in_db_using_offset (unexpected)
--- FAIL: BasicsTest#test_previously_changed_dup (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
--- FAIL: PersistenceTest#test_increment_nil_attribute (unexpected)
--- FAIL: BasicsTest#test_count_with_join (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_with_scopes (unexpected)
--- FAIL: InverseBelongsToTests#test_recursive_model_has_many_inversing (unexpected)
--- FAIL: PersistenceTest#test_update_column_with_one_changed_and_one_updated (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_0#test_create_join_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: BasicsTest#test_has_attribute_with_symbol (unexpected)
--- FAIL: PersistenceTest#test_update_for_record_with_only_primary_key (unexpected)
--- FAIL: BasicsTest#test_table_exists (unexpected)
--- FAIL: PersistenceTest#test_update_sti_subclass_type (unexpected)
--- FAIL: PersistenceTest#test_update_many_with_array_of_active_record_base_objects (unexpected)
--- FAIL: AssociationsExtensionsTest#test_association_with_default_scope (unexpected)
--- FAIL: BasicsTest#test_limit_should_take_value_from_latest_limit (unexpected)
--- FAIL: BasicsTest#test_bignum (unexpected)
--- FAIL: BasicsTest#test_all (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_decrement_counter_updates_custom_lock_version (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
--- FAIL: BasicsTest#test_equality_of_relation_and_collection_proxy (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_infers_column (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-sessions branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10061452?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10061452?buildTab=artifacts#/activerecord) on master @ [63eb6b86c1518fbed9db10e7d2afa1446b2b3b82](https://github.com/cockroachdb/cockroach/commits/63eb6b86c1518fbed9db10e7d2afa1446b2b3b82):


```
--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter_after_touch (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter (unexpected)
--- FAIL: BelongsToAssociationsTest#test_with_different_class_name (unexpected)
--- FAIL: BelongsToAssociationsTest#test_with_select (unexpected)
--- FAIL: BelongsToAssociationsTest#test_new_record_with_foreign_key_but_no_object (unexpected)
--- FAIL: BelongsToAssociationsTest#test_setting_foreign_key_after_nil_target_loaded (unexpected)
--- FAIL: BelongsToAssociationsTest#test_multiple_counter_cache_with_after_create_update (unexpected)
--- FAIL: BelongsToAssociationsTest#test_dependent_delete_and_destroy_with_belongs_to (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_works_with_model_called_Record (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: DefaultStringsTest#test_default_strings (unexpected)
--- FAIL: BelongsToAssociationsTest#test_should_set_foreign_key_on_create_association! (unexpected)
--- FAIL: BelongsToAssociationsTest#test_dependency_should_halt_parent_destruction_with_cascaded_three_levels (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_on_multiple_records (unexpected)
--- FAIL: BelongsToAssociationsTest#test_natural_assignment_with_primary_key (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_and_reassigned_parent (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_primary_key (unexpected)
--- FAIL: BelongsToAssociationsTest#test_build_with_block (unexpected)
--- FAIL: TouchLaterTest#test_touch_later_an_association_dont_autosave_parent (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_primary_key_counter (unexpected)
--- FAIL: BelongsToAssociationsTest#test_attributes_are_being_set_when_initialized_from_belongs_to_association_with_where_clause (unexpected)
--- FAIL: BelongsToAssociationsTest#test_assignment_updates_foreign_id_field_for_new_and_saved_records (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_counter_cache (unexpected)
--- FAIL: BelongsToAssociationsTest#test_create_with_block (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_id_assigning (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_change_from_one_persisted_record_to_another (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_foreign_key_type_string (unexpected)
--- FAIL: BelongsToAssociationsTest#test_reassigning_the_parent_id_updates_the_object (unexpected)
--- FAIL: BelongsToAssociationsTest#test_custom_attribute_with_select (unexpected)
--- FAIL: TouchLaterTest#test_touch_later_raise_if_non_persisted (unexpected)
--- FAIL: BelongsToAssociationsTest#test_create_bang_with_conditions (unexpected)
--- FAIL: BelongsToAssociationsTest#test_missing_attribute_error_is_raised_when_no_foreign_key_attribute (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_suffix (unexpected)
--- FAIL: BelongsToAssociationsTest#test_not_optional_relation (unexpected)
--- FAIL: BelongsToAssociationsTest#test_destroying_child_with_unloaded_parent_and_foreign_key_and_touch_is_possible_with_has_many_inversing (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_change_from_persisted_record_to_nil (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_reassign_with_namespaced_models_and_counters (unexpected)
--- FAIL: TouchLaterTest#test_touch_later_respects_no_touching_policy (unexpected)
--- FAIL: BelongsToAssociationsTest#test_creating_the_belonging_object (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_update (unexpected)
--- FAIL: TouchLaterTest#test_touch_touches_immediately (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_with_nil (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_polymorphic_changes (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_without_updated_at_attributes (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_with_custom_foreign_type (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10104121?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10104121?buildTab=artifacts#/activerecord) on master @ [107ad579b1ff8311ac4f4df4783985010a0322f9](https://github.com/cockroachdb/cockroach/commits/107ad579b1ff8311ac4f4df4783985010a0322f9):


```
--- FAIL: RelationTest#test_where_relation_with_delegated_ar_object (unexpected)
--- FAIL: RelationTest#test_any (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_add_remove_single_field_using_string_arguments (unexpected)
--- FAIL: FinderTest#test_member_on_unloaded_relation_with_limit (unexpected)
--- FAIL: FinderTest#test_fifth_with_offset (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
--- FAIL: AbsenceValidationTest#test_non_association (unexpected)
--- FAIL: FoxyFixturesTest#test_supports_polymorphic_belongs_to (unexpected)
--- FAIL: RelationTest#test_unscope_with_arel_sql (unexpected)
--- FAIL: EachTest#test_in_batches_should_return_all_if_the_limit_is_greater_than_the_number_of_records_when_load_is_true (unexpected)
--- FAIL: RelationTest#test_references_with_blank_value (unexpected)
--- FAIL: FinderTest#test_first_failing (unexpected)
--- FAIL: PostgresqlNumberTest#test_data_type (unexpected)
--- FAIL: FinderTest#test_find_by!_raises_RecordNotFound_if_the_record_is_missing (unexpected)
--- FAIL: RelationTest#test_unscope_with_unknown_column (unexpected)
--- FAIL: RelationTest#test_create_or_find_by (unexpected)
--- FAIL: EnumTest#test_validate_uniqueness (unexpected)
--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_strict_loading_has_one_relation (unexpected)
--- FAIL: RelationTest#test_loading_query_is_annotated_in_#inspect (unexpected)
--- FAIL: CockroachDB::AdapterForeignKeyTest#test_foreign_key_violations_on_delete_are_translated_to_specific_exception (unexpected)
--- FAIL: FinderTest#test_last_on_loaded_relation_should_not_use_sql (unexpected)
--- FAIL: RelationTest#test_order_triggers_eager_loading_when_ordering_using_symbols (unexpected)
--- FAIL: RelationScopingTest#test_circular_joins_with_scoping_does_not_crash (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_nonpreload_eager_loading_through_polymorphic_with_more_than_one_through_record (unexpected)
--- FAIL: ActiveRecord::FixtureSet::FileTest#test_names (unexpected)
--- FAIL: RelationTest#test_pluck_with_from_includes_quoted_original_table_name (unexpected)
--- FAIL: CounterCacheTest#test_counter_caches_are_updated_in_memory_when_the_default_value_is_nil (unexpected)
--- FAIL: RelationTest#test_automatically_added_having_references (unexpected)
--- FAIL: QueryLogsTest#test_select_is_commented (unexpected)
--- FAIL: RelationTest#test_finding_with_subquery_without_select_does_not_change_the_select (unexpected)
--- FAIL: RelationScopingTest#test_scoped_unscoped (unexpected)
--- FAIL: CounterCacheTest#test_reset_multiple_counters_with_touch:_true (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_empty_array_with_aggregate_having_multiple_mappings (unexpected)
--- FAIL: RelationTest#test_find_by!_with_hash_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: FoxyFixturesTest#test_strips_DEFAULTS_key (unexpected)
--- FAIL: FinderTest#test_exists_with_large_number (unexpected)
--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_timestamps_have_null_constraints_if_not_present_in_migration_of_create_table (unexpected)
--- FAIL: RelationTest#test_grouping_by_column_with_reserved_name (unexpected)
--- FAIL: EachTest#test_find_in_batches_should_use_any_column_as_primary_key (unexpected)
--- FAIL: RelationTest#test_select_with_subquery_string_in_from_does_not_use_original_table_name (unexpected)
--- FAIL: RelationScopingTest#test_scoping_applies_to_reload_with_all_queries (unexpected)
--- FAIL: RelationScopingTest#test_double_reverse_order_produces_original_order (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_through_belongs_to_after_destroy (unexpected)
--- FAIL: FinderTest#test_include_on_unloaded_relation_with_offset (unexpected)
--- FAIL: RelationScopingTest#test_scoped_create_with_where_with_array (unexpected)
--- FAIL: RelationTest#test_finding_with_asc_order_with_string (unexpected)
--- FAIL: EachTest#test_in_batches_if_not_loaded_executes_more_queries (unexpected)
--- FAIL: RelationTest#test_unscope_with_merge (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10122132?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10122132?buildTab=artifacts#/activerecord) on master @ [76078816b69ee6d1f8dacd581ef464660fb1b697](https://github.com/cockroachdb/cockroach/commits/76078816b69ee6d1f8dacd581ef464660fb1b697):


```
--- FAIL: HasAndBelongsToManyAssociationsTest#test_custom_join_table (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnsTest#test_change_column_nullability (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_of_multiple_records (unexpected)
--- FAIL: BasicsTest#test_benchmark_with_log_level (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_associate_with_create_bang_and_valid_options (unexpected)
--- FAIL: BasicsTest#test_preserving_time_objects_with_time_with_zone_conversion_to_default_timezone_utc (unexpected)
--- FAIL: FinderTest#test_last_with_irreversible_order (unexpected)
--- FAIL: FinderTest#test_second (unexpected)
--- FAIL: BasicsTest#test_initialize_with_invalid_attribute (unexpected)
--- FAIL: PostgresqlBitStringTest#test_assigning_invalid_hex_string_raises_exception (unexpected)
--- FAIL: PersistenceTest#test_find_via_reload (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_1#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: FinderTest#test_first_have_determined_order_by_default (unexpected)
--- FAIL: FinderTest#test_exists_with_aggregate_having_three_mappings_with_one_difference (unexpected)
--- FAIL: PersistenceTest#test_delete_isnt_affected_by_scoping (unexpected)
--- FAIL: FinderTest#test_member_on_unloaded_relation_with_offset (unexpected)
--- FAIL: BasicsTest#test_ignored_columns_not_included_in_SELECT (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_dynamic_find_all_should_respect_readonly_access (unexpected)
--- FAIL: PersistenceTest#test_delete_many (unexpected)
--- FAIL: CommentTest#test_blank_columns_created_in_block (unexpected)
--- FAIL: PersistenceTest#test_becomes_errors_base (unexpected)
--- FAIL: BasicsTest#test_last (unexpected)
--- FAIL: FinderTest#test_exists_with_string (unexpected)
--- FAIL: FinderTest#test_exists_with_order (unexpected)
--- FAIL: BasicsTest#test_#connected_to_many_cannot_be_called_with_classes_that_include_ActiveRecord::Base (unexpected)
--- FAIL: FinderTest#test_first_failing (unexpected)
--- FAIL: RelationScopingTest#test_scope_select_concatenates (unexpected)
--- FAIL: FinderTest#test_sole_failing_many (unexpected)
--- FAIL: FinderTest#test_fifth (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_removing_associations_on_destroy (unexpected)
--- FAIL: RelationScopingTest#test_scoped_create_with_where_with_array (unexpected)
--- FAIL: BasicsTest#test_null_fields (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_saving_multiple_relationships (unexpected)
--- FAIL: FinderTest#test_named_bind_variables_with_quotes (unexpected)
--- FAIL: BasicsTest#test_comparison_with_different_objects_in_array (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_deleting_array (unexpected)
--- FAIL: BasicsTest#test_cannot_call_connected_to_on_the_abstract_class_that_did_not_establish_the_connection (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_attributes_are_being_set_when_initialized_from_habtm_association_with_multiple_where_clauses (unexpected)
--- FAIL: FinderTest#test_include_on_unloaded_relation_with_match (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_third_to_last_bang (unexpected)
--- FAIL: PersistenceTest#test_update_columns_with_model_having_primary_key_other_than_id (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_:written_on (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_distinct_before_the_fact (unexpected)
--- FAIL: PersistenceTest#test_update_many! (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_new_with_values_in_collection (unexpected)
--- FAIL: RelationScopingTest#test_reverse_order_with_multiple_arel_nodes (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_explicit_table_name_and_aggregate (unexpected)
--- FAIL: CounterCacheTest#test_reset_multiple_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10137655?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10137655?buildTab=artifacts#/activerecord) on master @ [8f10f78aed7606edd454e886183bf22c74a3153e](https://github.com/cockroachdb/cockroach/commits/8f10f78aed7606edd454e886183bf22c74a3153e):


```
--- FAIL: CounterCacheTest#test_counter_caches_are_updated_in_memory_when_the_default_value_is_nil (unexpected)
--- FAIL: ReflectionTest#test_association_reflection_in_modules (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many_with_belongs_to (unexpected)
--- FAIL: CalculationsTest#test_calculation_grouped_by_association_doesnt_error_when_no_records_have_association (unexpected)
--- FAIL: QueryLogsTest#test_escaping_bad_comments (unexpected)
--- FAIL: DDLTest#test_reload_dumped_schema (unexpected)
--- FAIL: ValidationsTest#test_throw_away_typing (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_belongs_to (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_replace_with_less (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_adding_from_the_project_fixed_timestamp (unexpected)
--- FAIL: RelationScopingTest#test_scoping_respects_current_class (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_new_aliased_to_build (unexpected)
--- FAIL: CalculationsTest#test_distinct_count_with_order_and_offset (unexpected)
--- FAIL: RelationScopingTest#test_scoping_with_klass_method_works_in_the_scope_block (unexpected)
--- FAIL: CalculationsTest#test_minimum_and_maximum_on_time_attributes (unexpected)
--- FAIL: CalculationsTest#test_pluck_type_cast_with_eager_load_without_table_name_qualified_column (unexpected)
--- FAIL: ReflectionTest#test_columns_are_returned_in_the_order_they_were_declared (unexpected)
--- FAIL: RelationScopingTest#test_circular_joins_with_scoping_does_not_crash (unexpected)
--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_validate_presence_of_parent_works_with_inverse_of (unexpected)
--- FAIL: ValidationsTest#test_numericality_validation_with_mutation (unexpected)
--- FAIL: ReflectionTest#test_foreign_type (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_does_call_updated_at_when_it_is_assigned_via_a_string (unexpected)
--- FAIL: CalculationsTest#test_sum_with_not_auto_table_name_prefix_if_column_included (unexpected)
--- FAIL: CounterCacheTest#test_reset_multiple_counters (unexpected)
--- FAIL: ReflectionTest#test_foreign_key_is_inferred_from_model_name (unexpected)
--- FAIL: CalculationsTest#test_should_return_type_casted_values_with_group_and_expression (unexpected)
--- FAIL: CalculationsTest#test_should_not_use_alias_for_grouped_field (unexpected)
--- FAIL: CalculationsTest#test_pick_delegate_to_all (unexpected)
--- FAIL: DDLTest#test_create_point_geometry (unexpected)
--- FAIL: ReflectionTest#test_active_record_primary_key_raises_when_missing_primary_key (unexpected)
--- FAIL: SuppressorTest#test_suppresses_create_in_callback (unexpected)
--- FAIL: CalculationsTest#test_should_average_arel_attribute (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_should_property_quote_string_primary_keys (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_new_with_values_in_collection (unexpected)
--- FAIL: CalculationsTest#test_should_generate_valid_sql_with_joins_and_group (unexpected)
--- FAIL: CalculationsTest#test_pluck_with_selection_clause (unexpected)
--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_numeric_column_changes_from_zero_to_no_empty_string (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_find_in_association_with_options (unexpected)
--- FAIL: DDLTest#test_create_simple_geometry_using_shortcut (unexpected)
--- FAIL: ActiveRecord::PostgresqlConnectionTest#test_set_session_timezone (unexpected)
--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_should_refresh_saved_records_when_not_overwriting_unsaved_updates (unexpected)
--- FAIL: ValidationsTest#test_valid_using_special_context (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_polymorphic_add_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_dynamic_find_all_should_respect_readonly_access (unexpected)
--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_should_automatically_build_new_associated_models_for_each_entry_in_a_hash_where_the_id_is_missing (unexpected)
--- FAIL: DDLTest#test_null_constraints (unexpected)
--- FAIL: QueryLogsTest#test_delete_is_commented (unexpected)
--- FAIL: ValidationsTest#test_validators (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10155216?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10155216?buildTab=artifacts#/activerecord) on master @ [1df2f0e8dd222a21d799c63893fed65cb6a8ba99](https://github.com/cockroachdb/cockroach/commits/1df2f0e8dd222a21d799c63893fed65cb6a8ba99):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-1df2f0e8dd222a21d799c63893fed65cb6a8ba99
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10169345?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10169345?buildTab=artifacts#/activerecord) on master @ [a226099dfee6ac0159c0c40a54aaa753d646cf6e](https://github.com/cockroachdb/cockroach/commits/a226099dfee6ac0159c0c40a54aaa753d646cf6e):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-a226099dfee6ac0159c0c40a54aaa753d646cf6e
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10177814?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10177814?buildTab=artifacts#/activerecord) on master @ [b34c137004f111b915fb346da9a4770264d6c22f](https://github.com/cockroachdb/cockroach/commits/b34c137004f111b915fb346da9a4770264d6c22f):


```
--- FAIL: FinderTest#test_symbols_table_ref (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_qualified_attribute_dot_notation_string (unexpected)
--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_datetime_doesnt_set_precision_on_change_table_5_1 (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_array_of_ranges (unexpected)
--- FAIL: FinderTest#test_condition_local_time_interpolation_with_default_timezone_utc (unexpected)
--- FAIL: FinderTest#test_select_value (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_explicit_table_name_and_aggregate (unexpected)
--- FAIL: FinderTest#test_exists_with_left_joins (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_on_existing_relation (unexpected)
--- FAIL: FinderTest#test_exists_with_aggregate_having_three_mappings (unexpected)
--- FAIL: FinderTest#test_member_on_loaded_relation_with_match (unexpected)
--- FAIL: FinderTest#test_finder_with_offset_string (unexpected)
--- FAIL: FinderTest#test_exists_with_string (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_second_to_last_bang (unexpected)
--- FAIL: FinderTest#test_eager_load_for_no_has_many_with_limit_and_left_joins_for_has_many (unexpected)
--- FAIL: FinderTest#test_find_by_with_range_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: FinderTest#test_exists_with_eager_load (unexpected)
--- FAIL: FinderTest#test_exists_passing_active_record_object_is_not_permitted (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_hashed_table_name (unexpected)
--- FAIL: FinderTest#test_find_by_nil_attribute (unexpected)
--- FAIL: FinderTest#test_find_by_id_with_conditions_with_or (unexpected)
--- FAIL: FinderTest#test_member_on_loaded_relation_without_match (unexpected)
--- FAIL: FinderTest#test_sole (unexpected)
--- FAIL: FinderTest#test_include_on_loaded_relation_with_match (unexpected)
--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_timestamps_doesnt_set_precision_on_create_table (unexpected)
--- FAIL: FinderTest#test_find_with_large_number (unexpected)
--- FAIL: FinderTest#test_find_on_association_proxy_conditions (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_nullify_foreign_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: FinderTest#test_find_by_doesn't_have_implicit_ordering (unexpected)
--- FAIL: FinderTest#test_implicit_order_set_to_primary_key (unexpected)
--- FAIL: FinderTest#test_exists_with_empty_hash_arg (unexpected)
--- FAIL: ActiveRecord::AdapterTest#test_value_limit_violations_are_translated_to_specific_exception (unexpected)
--- FAIL: FinderTest#test_third (unexpected)
--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_datetime_doesnt_set_precision_on_change_table_6_0 (unexpected)
--- FAIL: FinderTest#test_take_and_first_and_last_with_integer_should_use_sql_limit (unexpected)
--- FAIL: FinderTest#test_last_with_irreversible_order (unexpected)
--- FAIL: FinderTest#test_last_with_integer_and_reorder_should_use_sql_limit (unexpected)
--- FAIL: FinderTest#test_find_by_with_associations (unexpected)
--- FAIL: FinderTest#test_find_by_title_and_id_with_hash (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
--- FAIL: FinderTest#test_find_with_ids_and_order_clause (unexpected)
--- FAIL: FinderTest#test_last_on_loaded_relation_should_not_use_sql (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_limit (unexpected)
--- FAIL: FinderTest#test_joins_with_string_array (unexpected)
--- FAIL: FinderTest#test_exists_returns_false_with_false_arg (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10186190?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10186190?buildTab=artifacts#/activerecord) on master @ [b34c137004f111b915fb346da9a4770264d6c22f](https://github.com/cockroachdb/cockroach/commits/b34c137004f111b915fb346da9a4770264d6c22f):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-b34c137004f111b915fb346da9a4770264d6c22f
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10200554?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10200554?buildTab=artifacts#/activerecord) on master @ [a1b8517dd93d386fdf6b170982c604e6396b50e9](https://github.com/cockroachdb/cockroach/commits/a1b8517dd93d386fdf6b170982c604e6396b50e9):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-a1b8517dd93d386fdf6b170982c604e6396b50e9
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10236439?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10236439?buildTab=artifacts#/activerecord) on master @ [5d944f53668fedf017e0324475c408c3f14ae32e](https://github.com/cockroachdb/cockroach/commits/5d944f53668fedf017e0324475c408c3f14ae32e):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-5d944f53668fedf017e0324475c408c3f14ae32e
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10253356?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10253356?buildTab=artifacts#/activerecord) on master @ [54058c8455ca84f6734890dae883f535b7d75104](https://github.com/cockroachdb/cockroach/commits/54058c8455ca84f6734890dae883f535b7d75104):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-54058c8455ca84f6734890dae883f535b7d75104
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10271808?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10271808?buildTab=artifacts#/activerecord) on master @ [9d6e7baebdc0d8b1a57ce8dc158fec35b48e9e2a](https://github.com/cockroachdb/cockroach/commits/9d6e7baebdc0d8b1a57ce8dc158fec35b48e9e2a):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-9d6e7baebdc0d8b1a57ce8dc158fec35b48e9e2a
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10281147?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10281147?buildTab=artifacts#/activerecord) on master @ [dd8dbfc4fa61763dc4fa8fbc19d3a7f2a30bb1e2](https://github.com/cockroachdb/cockroach/commits/dd8dbfc4fa61763dc4fa8fbc19d3a7f2a30bb1e2):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-dd8dbfc4fa61763dc4fa8fbc19d3a7f2a30bb1e2
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10301329?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10301329?buildTab=artifacts#/activerecord) on master @ [e48d8a376760498c20aa922ddd3b52b5b31dcb19](https://github.com/cockroachdb/cockroach/commits/e48d8a376760498c20aa922ddd3b52b5b31dcb19):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-e48d8a376760498c20aa922ddd3b52b5b31dcb19
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_number_in_scientific_notation (unexpected)
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10321116?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10321116?buildTab=artifacts#/activerecord) on master @ [906e7ade9d3891a974ab02e89b4cb69c35ffda13](https://github.com/cockroachdb/cockroach/commits/906e7ade9d3891a974ab02e89b4cb69c35ffda13):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.1.0-alpha.8-dev-906e7ade9d3891a974ab02e89b4cb69c35ffda13
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10338231?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10338231?buildTab=artifacts#/activerecord) on master @ [e2b704df25b012baf7d5fbe26e034f520a70f25c](https://github.com/cockroachdb/cockroach/commits/e2b704df25b012baf7d5fbe26e034f520a70f25c):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-e2b704df25b012baf7d5fbe26e034f520a70f25c
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10356406?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10356406?buildTab=artifacts#/activerecord) on master @ [bf25b7bf93a6b8e57557813957da263389796bb4](https://github.com/cockroachdb/cockroach/commits/bf25b7bf93a6b8e57557813957da263389796bb4):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-bf25b7bf93a6b8e57557813957da263389796bb4
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
2 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- FAIL: ActiveRecord::Encryption::EncryptionPerformanceTest#test_performance_when_saving_records (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10409513?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10409513?buildTab=artifacts#/activerecord) on master @ [530e2ef6d3e974202b2015c22c7ea9eb1c20a29e](https://github.com/cockroachdb/cockroach/commits/530e2ef6d3e974202b2015c22c7ea9eb1c20a29e):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-530e2ef6d3e974202b2015c22c7ea9eb1c20a29e
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10446921?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10446921?buildTab=artifacts#/activerecord) on master @ [c7a7e423506c88f655c08d6ba6acd6e907e1dae5](https://github.com/cockroachdb/cockroach/commits/c7a7e423506c88f655c08d6ba6acd6e907e1dae5):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-c7a7e423506c88f655c08d6ba6acd6e907e1dae5
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10464260?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10464260?buildTab=artifacts#/activerecord) on master @ [33525d55ab1f2e1973a8788afe40bae2ed8c55a5](https://github.com/cockroachdb/cockroach/commits/33525d55ab1f2e1973a8788afe40bae2ed8c55a5):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-33525d55ab1f2e1973a8788afe40bae2ed8c55a5
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10477887?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10477887?buildTab=artifacts#/activerecord) on master @ [61806f42e4c833b863ca9c2f62ce34918f1c8277](https://github.com/cockroachdb/cockroach/commits/61806f42e4c833b863ca9c2f62ce34918f1c8277):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-61806f42e4c833b863ca9c2f62ce34918f1c8277
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10484309?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10484309?buildTab=artifacts#/activerecord) on master @ [6e38f676273d08b3a111213d6c9f7945629228d7](https://github.com/cockroachdb/cockroach/commits/6e38f676273d08b3a111213d6c9f7945629228d7):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-6e38f676273d08b3a111213d6c9f7945629228d7
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10491070?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10491070?buildTab=artifacts#/activerecord) on master @ [9c2f65094e2aa121d8e9b15c90e4d657848f1667](https://github.com/cockroachdb/cockroach/commits/9c2f65094e2aa121d8e9b15c90e4d657848f1667):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-9c2f65094e2aa121d8e9b15c90e4d657848f1667
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10522175?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10522175?buildTab=artifacts#/activerecord) on master @ [13addf52fe9b979f5cc3aeabac518058ec7715ff](https://github.com/cockroachdb/cockroach/commits/13addf52fe9b979f5cc3aeabac518058ec7715ff):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-13addf52fe9b979f5cc3aeabac518058ec7715ff
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10539670?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10539670?buildTab=artifacts#/activerecord) on master @ [3c55b81a4e64b62b7a1a59fad76c8d8a39f867a0](https://github.com/cockroachdb/cockroach/commits/3c55b81a4e64b62b7a1a59fad76c8d8a39f867a0):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-3c55b81a4e64b62b7a1a59fad76c8d8a39f867a0
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10556171?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10556171?buildTab=artifacts#/activerecord) on master @ [41a817f0d4a22f921e3d291f08394eaa8bce9adb](https://github.com/cockroachdb/cockroach/commits/41a817f0d4a22f921e3d291f08394eaa8bce9adb):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-41a817f0d4a22f921e3d291f08394eaa8bce9adb
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10571450?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10571450?buildTab=artifacts#/activerecord) on master @ [795dd78a4c02f04184192b61c30bcf8fd980b5ea](https://github.com/cockroachdb/cockroach/commits/795dd78a4c02f04184192b61c30bcf8fd980b5ea):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-795dd78a4c02f04184192b61c30bcf8fd980b5ea
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10578490?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10578490?buildTab=artifacts#/activerecord) on master @ [795dd78a4c02f04184192b61c30bcf8fd980b5ea](https://github.com/cockroachdb/cockroach/commits/795dd78a4c02f04184192b61c30bcf8fd980b5ea):


```
--- FAIL: HabtmDestroyOrderTest#test_not_destroying_a_student_with_lessons_leaves_student<=>lesson_association_intact (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_destroy (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_and_reassigned_parent (unexpected)
--- FAIL: SerializedAttributeTest#test_decorated_type_with_type_for_attribute (unexpected)
--- FAIL: FixturesTest#test_fixtures_from_root_yml_with_instantiation (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_should_raise_exception_on_assignment_with_wrong_type (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_boolean_value_true (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_proxy_should_not_respond_to_private_methods (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_change_from_persisted_record_to_new_record (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_invalid_dependent_option_raises_exception (unexpected)
--- FAIL: BelongsToAssociationsTest#test_replace_counter_cache (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_setting_foreign_key_after_nil_target_loaded (unexpected)
--- FAIL: BelongsToAssociationsTest#test_creating_the_belonging_object (unexpected)
--- FAIL: OptimisticLockingTest#test_update_with_lock_version_without_default_should_work_on_dirty_value_before_type_cast (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_primary_key_counter (unexpected)
--- FAIL: OptimisticLockingTest#test_update_without_attributes_does_not_only_update_lock_version (unexpected)
--- FAIL: BelongsToAssociationsTest#test_natural_assignment_to_nil (unexpected)
--- FAIL: DefaultScopingTest#test_with_abstract_class_where_clause_should_not_be_duplicated (unexpected)
--- FAIL: BidirectionalDestroyDependenciesTest#test_bidirectional_dependence_when_destroying_item_with_has_one_association (unexpected)
--- FAIL: ActiveRecordSchemaTest#test_timestamps_without_null_set_null_to_false_on_change_table (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_new_when_explicitly_passing_value (unexpected)
--- FAIL: TouchLaterTest#test_touching_three_deep (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_with_class_constraint (unexpected)
--- FAIL: FixturesTest#test_yaml_file_with_invalid_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: BelongsToAssociationsTest#test_attributes_are_set_without_error_when_initialized_from_belongs_to_association_with_array_in_where_clause (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
--- FAIL: ActiveRecord::Encryption::EncryptableRecordTest#test_encrypts_serialized_attributes (unexpected)
--- FAIL: FixturesTest#test_complete_instantiation (unexpected)
--- FAIL: BelongsToAssociationsTest#test_build_with_conditions (unexpected)
--- FAIL: AttributeMethodsTest#test_respond_to?_with_a_custom_primary_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_an_unrelated_type (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_singular_ids_are_reloaded_after_collection_concat (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_validation_does_not_validate_non_dirty_association_target (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_primary_key_auto_assigns_on_save (unexpected)
--- FAIL: FixturesTest#test_instantiation (unexpected)
--- FAIL: BelongsToAssociationsTest#test_reloading_association_with_key_change (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_with_false (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_empty_update (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_with_custom_name_touch_old_belongs_to_model (unexpected)
--- FAIL: TouchLaterTest#test_touch_touches_immediately_with_a_custom_time (unexpected)
--- FAIL: BelongsToAssociationsTest#test_counter_cache (unexpected)
--- FAIL: FixturesTest#test_insert_with_datetime (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10585330?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10585330?buildTab=artifacts#/activerecord) on master @ [9633594f139d80d1555127a09f1ecd12dd1d9b11](https://github.com/cockroachdb/cockroach/commits/9633594f139d80d1555127a09f1ecd12dd1d9b11):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-9633594f139d80d1555127a09f1ecd12dd1d9b11
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10594790?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10594790?buildTab=artifacts#/activerecord) on master @ [76da6c7dfb4d71ad614715ec5e230a6cc76fbd0e](https://github.com/cockroachdb/cockroach/commits/76da6c7dfb4d71ad614715ec5e230a6cc76fbd0e):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-76da6c7dfb4d71ad614715ec5e230a6cc76fbd0e
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10611064?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10611064?buildTab=artifacts#/activerecord) on master @ [1eb628e8c8a7eb1cbf9bfa2bd6c31982c25cbba0](https://github.com/cockroachdb/cockroach/commits/1eb628e8c8a7eb1cbf9bfa2bd6c31982c25cbba0):


```
--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_do_not_require_a_type (unexpected)
--- FAIL: OptimisticLockingTest#test_touch_existing_lock (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_new (unexpected)
--- FAIL: PostgresqlBitStringTest#test_roundtrip (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_destroy_stale_object (unexpected)
--- FAIL: AssociationProxyTest#test_proxy_object_is_cached (unexpected)
--- FAIL: AssociationProxyTest#test_push_followed_by_save_does_not_load_target (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_infers_column (unexpected)
--- FAIL: InsertAllTest#test_insert_all_and_upsert_all_works_with_composite_primary_keys_when_unique_by_is_provided (unexpected)
--- FAIL: LegacyPrimaryKeyTest::V4_2#test_legacy_join_table_column_options_should_be_overwritten (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_not_backed_by_database_columns_appear_in_inspect (unexpected)
--- FAIL: OptimisticLockingTest#test_touch_existing_lock_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: OptimisticLockingTest#test_update_without_attributes_does_not_only_update_lock_version (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_decrement_counter_updates_lock_version (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_changing_defaults (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_user_provided_defaults_are_persisted_even_if_unchanged (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_validation_does_not_validate_non_dirty_association_target (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_store_association_in_two_relations_with_one_save_in_existing_object_with_values (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_suffix (unexpected)
--- FAIL: ReadOnlyTest#test_cant_touch_readonly_record (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_update_counters_updates_custom_lock_version (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_symbol_column (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_without_default_queries_count (unexpected)
--- FAIL: AssociationProxyTest#test_target_merging_ignores_persisted_in_memory_records (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_overloading_types (unexpected)
--- FAIL: AssociationProxyTest#test_reset_unloads_target (unexpected)
--- FAIL: AssociationsTest#test_include_with_order_works (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
--- FAIL: OptimisticLockingTest#test_explicit_update_lock_column_raise_error (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_properties_assigned_in_constructor (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_1#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_store_association_in_two_relations_with_one_save_in_existing_object (unexpected)
--- FAIL: AssociationProxyTest#test_push_does_not_lose_additions_to_new_record (unexpected)
--- FAIL: PostgresqlNumberTest#test_update (unexpected)
--- FAIL: AssociationProxyTest#test_reload_returns_association (unexpected)
--- FAIL: PostgresqlIntegerTest#test_schema_properly_respects_bigint_ranges (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_defaults_are_not_touched_on_the_columns (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_stored_column (unexpected)
--- FAIL: AssociationProxyTest#test_first!_works_on_loaded_associations (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_new_when_explicitly_passing_nil (unexpected)
--- FAIL: NestedClassTest#test_nested_model (unexpected)
--- FAIL: OptimisticLockingTest#test_non_integer_lock_destroy (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_immutable_strings_by_default_changes_schema_inference_for_string_columns (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_1#test_change_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_increment_counter_updates_custom_lock_version (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10644056?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10644056?buildTab=artifacts#/activerecord) on master @ [cb6dd27fdfbfc8a9d0d4753809f4588bae8060d0](https://github.com/cockroachdb/cockroach/commits/cb6dd27fdfbfc8a9d0d4753809f4588bae8060d0):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-cb6dd27fdfbfc8a9d0d4753809f4588bae8060d0
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10657405?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10657405?buildTab=artifacts#/activerecord) on master @ [7fd4c21157221eae9e7d5892d89d2b5a671aba3e](https://github.com/cockroachdb/cockroach/commits/7fd4c21157221eae9e7d5892d89d2b5a671aba3e):


```
--- FAIL: RelationScopingTest#test_raises_error_if_all_queries_is_set_to_false_while_nested (unexpected)
--- FAIL: BasicsTest#test_column_types_typecast (unexpected)
--- FAIL: BasicsTest#test_find_keeps_multiple_group_values (unexpected)
--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_conditions_on_association (unexpected)
--- FAIL: HasOneAssociationsTest#test_dependence_with_nil_associate (unexpected)
--- FAIL: BasicsTest#test_abstract_class_table_name (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_association (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_default_scope_as_class_method_using_find_by_method (unexpected)
--- FAIL: EagerAssociationTest#test_finding_with_includes_on_null_belongs_to_polymorphic_association (unexpected)
--- FAIL: EagerAssociationTest#test_conditions_on_join_table_with_include_and_limit (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_respects_hash_condition (unexpected)
--- FAIL: BasicsTest#test_destroy_without_prepared_statement (unexpected)
--- FAIL: BasicsTest#test_.columns_hash_raises_an_error_if_the_record_has_an_empty_table_name (unexpected)
--- FAIL: EagerAssociationTest#test_deep_including_through_habtm (unexpected)
--- FAIL: HasOneAssociationsTest#test_building_the_associated_object_with_an_invalid_type (unexpected)
--- FAIL: LintTest#test_to_param (unexpected)
--- FAIL: RelationScopingTest#test_scope_select_concatenates (unexpected)
--- FAIL: BasicsTest#test_benchmark_with_log_level (unexpected)
--- FAIL: EagerAssociationTest#test_limited_eager_with_numeric_in_association (unexpected)
--- FAIL: BasicsTest#test_time_zone_aware_attribute_with_default_timezone_utc_on_utc_can_be_created (unexpected)
--- FAIL: OptimisticLockingTest#test_destroy_with_dirty_primary_key (unexpected)
--- FAIL: HasOneAssociationsTest#test_attributes_are_being_set_when_initialized_from_has_one_association_with_where_clause (unexpected)
--- FAIL: RelationScopingTest#test_reverse_order_with_arel_node (unexpected)
--- FAIL: BasicsTest#test_set_table_name_symbol_converted_to_string (unexpected)
--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_order_on_nested_association (unexpected)
--- FAIL: FoxyFixturesTest#test_identifies_consistently (unexpected)
--- FAIL: AssociationsExtensionsTest#test_marshalling_named_extensions (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_has_one_dependent_does_not_destroy_dependent (unexpected)
--- FAIL: HasOneAssociationsTest#test_reload_association (unexpected)
--- FAIL: BasicsTest#test_group_weirds_by_from (unexpected)
--- FAIL: RelationScopingTest#test_scoping_with_query_method_works_in_the_scope_block (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_has_many_and_limit_and_conditions_array_on_the_eagers (unexpected)
--- FAIL: EagerAssociationTest#test_joins_with_includes_should_preload_via_joins (unexpected)
--- FAIL: EagerAssociationTest#test_eager_load_has_many_through_with_string_keys (unexpected)
--- FAIL: BasicsTest#test_clone_of_new_object_marks_attributes_as_dirty (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_polymorphic_add_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: AssociationsTest#test_should_construct_new_finder_sql_after_create (unexpected)
--- FAIL: EagerAssociationTest#test_scoping_with_a_circular_preload (unexpected)
--- FAIL: FoxyFixturesTest#test_populates_timestamp_columns (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_default_scope_as_callable (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_invalid_association_reference (unexpected)
--- FAIL: HasOneAssociationsTest#test_association_keys_bypass_attribute_protection (unexpected)
--- FAIL: FoxyFixturesTest#test_supports_sti_with_respective_files (unexpected)
--- FAIL: BasicsTest#test_comparison_with_different_objects (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_floating_point_numbers (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_limit_and_conditions (unexpected)
--- FAIL: EagerAssociationTest#test_preconfigured_includes_with_belongs_to (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10664692?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10664692?buildTab=artifacts#/activerecord) on master @ [a2c2c060a423ee410b57868e657df644f2619cb3](https://github.com/cockroachdb/cockroach/commits/a2c2c060a423ee410b57868e657df644f2619cb3):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-a2c2c060a423ee410b57868e657df644f2619cb3
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10671650?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10671650?buildTab=artifacts#/activerecord) on master @ [a2c2c060a423ee410b57868e657df644f2619cb3](https://github.com/cockroachdb/cockroach/commits/a2c2c060a423ee410b57868e657df644f2619cb3):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-a2c2c060a423ee410b57868e657df644f2619cb3
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10689032?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10689032?buildTab=artifacts#/activerecord) on master @ [0b0a212ecbe514b55301f4a963edc92b16de9bab](https://github.com/cockroachdb/cockroach/commits/0b0a212ecbe514b55301f4a963edc92b16de9bab):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-0b0a212ecbe514b55301f4a963edc92b16de9bab
Tests run against activerecord 7.0.3
7163 Total Tests Run
7154 tests passed
9 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PessimisticLockingTest#test_with_lock_sets_isolation (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10771334?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10771334?buildTab=artifacts#/activerecord) on master @ [428dc9da6a320de218460de6c6c8807caa4ded98](https://github.com/cockroachdb/cockroach/commits/428dc9da6a320de218460de6c6c8807caa4ded98):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-428dc9da6a320de218460de6c6c8807caa4ded98
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionPoolTest#test_checkout_fairness_by_group (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10813231?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10813231?buildTab=artifacts#/activerecord) on master @ [1137aee0089fc2fe161bab0f1c899d2ddc997c90](https://github.com/cockroachdb/cockroach/commits/1137aee0089fc2fe161bab0f1c899d2ddc997c90):


```
--- FAIL: SerializedAttributeTest#test_json_read_legacy_null (unexpected)
--- FAIL: CounterCacheTest#test_increment_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: DDLTest#test_drop_geography_column (unexpected)
--- FAIL: StrictLoadingTest#test_raises_if_strict_loading_and_lazy_loading (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_by_counter_name (unexpected)
--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_timestamptz_if_datetime_type_changed (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type_cast_from_numeric (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_overloaded_properties_with_limit (unexpected)
--- FAIL: CockroachDB::AdapterForeignKeyTest#test_foreign_key_violations_on_delete_are_translated_to_specific_exception (unexpected)
--- FAIL: RelationScopingTest#test_scoping_applies_to_update_with_all_queries (unexpected)
--- FAIL: TimestampTest#test_touching_a_record_touches_polymorphic_record (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_destroying_many (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_last_preserves_scope (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_with_scopes (unexpected)
--- FAIL: HotCompatibilityTest#test_cleans_up_after_prepared_statement_failure_in_nested_transactions (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_last (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_on_alias_attribute (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_2#test_create_join_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnAHasOneAssociation#test_callbacks_firing_order_on_update (unexpected)
--- FAIL: RelationScopingTest#test_scoped_create_with_create_with (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_by_default_is_inheritable (unexpected)
--- FAIL: RequiredAssociationsTest#test_belongs_to_associations_can_be_optional_by_default (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_no_default_class_for_object (unexpected)
--- FAIL: RelationScopingTest#test_scoped_create_with_where_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_primary_key_auto_assigns_on_save (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_removing_associations_on_destroy (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_not_backed_by_database_columns_are_not_dirty_when_unchanged (unexpected)
--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_has_many_relation_if_strict_loading_by_default (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_joins (unexpected)
--- FAIL: PrimaryKeysTest#test_find_with_multiple_ids_should_quote_pkey (unexpected)
--- FAIL: SerializedAttributeTest#test_newly_emptied_serialized_hash_is_changed (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_is_ignored_in_validation_context (unexpected)
--- FAIL: PessimisticLockingTest#test_locking_in_after_save_callback (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_habtm_relation_if_strict_loading_by_default (unexpected)
--- FAIL: RelationScopingTest#test_update_all_default_scope_filters_on_joins (unexpected)
--- FAIL: PostgresqlTimestampTest#test_timestamp_with_zone_values_without_rails_time_zone_support (unexpected)
--- FAIL: DDLTest#test_create_simple_geography_using_shortcut (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_key_format_is_not_too_precise (unexpected)
--- FAIL: LintTest#test_model_naming (unexpected)
--- FAIL: CounterCacheTest#test_reset_multiple_counters_with_touch:_true (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: CockroachDB::FixturesResetPkSequenceTest#test_resets_to_min_pk_with_specified_pk_and_sequence (unexpected)
--- FAIL: RelationScopingTest#test_scoped_create_with_where_with_range (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_violation_logs_on_polymorphic_relation (unexpected)
--- FAIL: PostgresqlEnumTest#test_schema_load (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #101291 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations todo-deprecate.branch-release-23.1.0]
- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10837254?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10837254?buildTab=artifacts#/activerecord) on master @ [0207c613fa7c8f3ab66c4518ee1e52dabb863426](https://github.com/cockroachdb/cockroach/commits/0207c613fa7c8f3ab66c4518ee1e52dabb863426):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-0207c613fa7c8f3ab66c4518ee1e52dabb863426
Tests run against activerecord 7.0.3
7163 Total Tests Run
7137 tests passed
26 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
17 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_not_set (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_exception_on_removing_index_without_column_option (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert_change_table (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_up_only (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_down_with_table_name_prefix (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_the_select_one_on_the_same_table (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_legacy_down (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert_add_index_with_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert_change_table_comment (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10843924?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10843924?buildTab=artifacts#/activerecord) on master @ [0207c613fa7c8f3ab66c4518ee1e52dabb863426](https://github.com/cockroachdb/cockroach/commits/0207c613fa7c8f3ab66c4518ee1e52dabb863426):


```
--- FAIL: HasManyAssociationsTest#test_delete_all (unexpected)
--- FAIL: BasicsTest#test_successful_comparison_of_like_class_records (unexpected)
--- FAIL: BasicsTest#test_values_at (unexpected)
--- FAIL: HasManyAssociationsTest#test_update_all_on_association_accessed_before_save (unexpected)
--- FAIL: UpdateAllTest#test_update_all_with_joins_and_limit_and_order (unexpected)
--- FAIL: BelongsToAssociationsTest#test_eager_loading_with_primary_key (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_primary_key (unexpected)
--- FAIL: BasicsTest#test_hashing (unexpected)
--- FAIL: SerializedAttributeTest#test_nil_not_serialized_with_class_constraint (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute (unexpected)
--- FAIL: HasManyAssociationsTest#test_select_with_block_and_dirty_target (unexpected)
--- FAIL: BasicsTest#test_#connected_to_many_sets_prevent_writes_if_role_is_reading (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_boolean_value_false (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_collection_singular_ids_setter_with_string_primary_keys (unexpected)
--- FAIL: UpdateAllTest#test_update_on_relation_passing_active_record_object_is_not_permitted (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_singleton_has_many_through (unexpected)
--- FAIL: PostgresqlBitStringTest#test_bit_string_column (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_with_out_of_range_value (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_empty_with_counter_cache (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_associate_with_create_exclamation_and_no_options (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_reassignment_of_associated_type_updates_the_object (unexpected)
--- FAIL: BelongsToAssociationsTest#test_attributes_are_being_set_when_initialized_from_belongs_to_association_with_where_clause (unexpected)
--- FAIL: BasicsTest#test_cannot_call_connected_to_with_role_and_shard_on_non-abstract_classes (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_child_is_visible_to_join_model_in_add_association_callbacks (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_create_should_not_raise_exception_when_join_record_has_errors (unexpected)
--- FAIL: BasicsTest#test_slice (unexpected)
--- FAIL: HasManyAssociationsTest#test_finding_with_different_class_name_and_order (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_counter_cache (unexpected)
--- FAIL: BasicsTest#test_find_symbol_ordered_last (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_in_collection (unexpected)
--- FAIL: SerializedAttributeTest#test_where_by_serialized_attribute_with_array (unexpected)
--- FAIL: BasicsTest#test_table_name_based_on_model_name (unexpected)
--- FAIL: HasManyAssociationsTest#test_replace_returns_target (unexpected)
--- FAIL: HasManyAssociationsTest#test_collection_not_empty_after_building (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_with_scope_that_has_joined_same_table_with_parent_relation (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_with_custom_foreign_type (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_preloading_empty_through_association_via_joins (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_can_update_through_association (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_associations_on_new_records_use_null_relations (unexpected)
--- FAIL: HasManyAssociationsTest#test_defining_has_many_association_with_delete_all_dependency_lazily_evaluates_target_class (unexpected)
--- FAIL: HasManyAssociationsTest#test_finder_method_with_dirty_target (unexpected)
--- FAIL: BasicsTest#test_toggle_attribute (unexpected)
--- FAIL: HasManyAssociationsTest#test_counting_with_association_limit (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_association_with_same_foreign_key_name (unexpected)
--- FAIL: BasicsTest#test_records_of_different_classes_have_different_hashes (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_marshal_dump (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_one_should_return_false_if_more_than_one (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10858669?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10858669?buildTab=artifacts#/activerecord) on master @ [382a6efb7985b1993e0398c7b29f0e0b746587f4](https://github.com/cockroachdb/cockroach/commits/382a6efb7985b1993e0398c7b29f0e0b746587f4):


```
--- FAIL: EagerAssociationTest#test_eager_with_valid_association_as_string_not_symbol (unexpected)
--- FAIL: AutomaticInverseFindingTests#test_has_one_and_belongs_to_with_a_scope_and_automatic_scope_inversing_should_find_inverse_automatically (unexpected)
--- FAIL: EagerAssociationTest#test_eager_load_has_many_through_quotes_table_and_column_names (unexpected)
--- FAIL: InverseBelongsToTests#test_building_has_many_parent_association_inverses_one_record (unexpected)
--- FAIL: FoxyFixturesTest#test_supports_label_interpolation_for_integer_label (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_with_custom_primary_key_on_has_many_source (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_through_association_with_joins (unexpected)
--- FAIL: EagerAssociationTest#test_including_duplicate_objects_from_has_many (unexpected)
--- FAIL: AggregationsTest#test_gps_equality (unexpected)
--- FAIL: CalculationsTest#test_should_not_perform_joined_include_by_default (unexpected)
--- FAIL: CalculationsTest#test_minimum_and_maximum_on_non_numeric_type (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_loading_for_new_record (unexpected)
--- FAIL: FinderTest#test_find_by_with_range_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: AssociationsJoinModelTest#test_include_has_many_through (unexpected)
--- FAIL: AutomaticInverseFindingTests#test_has_many_and_belongs_to_should_find_inverse_automatically_for_sti (unexpected)
--- FAIL: CalculationsTest#test_count_with_too_many_parameters_raises (unexpected)
--- FAIL: CalculationsTest#test_minimum_and_maximum_on_tz_aware_attributes (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_open_ended_range (unexpected)
--- FAIL: AssociationsJoinModelTest#test_exceptions_have_suggestions_for_fix (unexpected)
--- FAIL: CalculationsTest#test_pluck_type_cast_with_joins_without_table_name_qualified_column (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_inferred_foreign_key_from_association_name (unexpected)
--- FAIL: FoxyFixturesTest#test_namespaced_models (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_by_integer_id (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_one_through_with_has_one_source_reflection (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_appending_on_disable_joins_through (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_polymorphic_has_many_through_when_through_association_has_not_loaded (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_going_through_join_model_with_custom_foreign_key (unexpected)
--- FAIL: FinderTest#test_exists_with_string (unexpected)
--- FAIL: AggregationsTest#test_allow_nil_address_set_to_nil (unexpected)
--- FAIL: HasManyAssociationsTest#test_counter_cache_updates_in_memory_after_update_with_inverse_of_enabled (unexpected)
--- FAIL: HasManyAssociationsTest#test_assign_ids_ignoring_blanks (unexpected)
--- FAIL: FinderTest#test_find_with_ids_where_and_limit (unexpected)
--- FAIL: EagerAssociationTest#test_preloading_empty_belongs_to (unexpected)
--- FAIL: ActiveRecord::RelationTest#test_empty_where_values_hash (unexpected)
--- FAIL: MultiParameterAttributeTest#test_multiparameter_assignment_of_aggregation_with_large_index (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_set_record_after_delete_association (unexpected)
--- FAIL: ReadOnlyTest#test_readonly_scoping (unexpected)
--- FAIL: InverseBelongsToTests#test_recursive_inverse_on_recursive_model_has_many_inversing (unexpected)
--- FAIL: ReflectionTest#test_never_validate_association_if_explicit (unexpected)
--- FAIL: ReflectionTest#test_has_many_reflection (unexpected)
--- FAIL: EagerAssociationTest#test_eager_loading_with_conditions_on_string_joined_table_preloads (unexpected)
--- FAIL: CalculationsTest#test_should_count_manual_select_with_include (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_array_of_ranges (unexpected)
--- FAIL: FinderTest#test_exists_with_includes_limit_and_empty_result (unexpected)
--- FAIL: EagerAssociationTest#test_associations_loaded_for_all_records (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_store_association_in_two_relations_with_one_save_in_existing_object_with_values (unexpected)
--- FAIL: CalculationsTest#test_from_option_with_specified_index (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10892350?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10892350?buildTab=artifacts#/activerecord) on master @ [8dcde9c4635f27cefdbc167729c95bf865d9dfd5](https://github.com/cockroachdb/cockroach/commits/8dcde9c4635f27cefdbc167729c95bf865d9dfd5):


```
(orm_helpers.go:212).summarizeFailed: 
Tests run on Cockroach v23.2.0-alpha.00000000-dev-8dcde9c4635f27cefdbc167729c95bf865d9dfd5
Tests run against activerecord 7.0.3
7163 Total Tests Run
7153 tests passed
10 tests failed
25 tests skipped
119 tests ignored
0 tests passed unexpectedly
1 test failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::Encryption::StoragePerformanceTest#test_storage_overload_without_storing_keys_is_acceptable (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10906783?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10906783?buildTab=artifacts#/activerecord) on master @ [ac6d73de8e27cae4e087bb42ee03bfc515939104](https://github.com/cockroachdb/cockroach/commits/ac6d73de8e27cae4e087bb42ee03bfc515939104):


```
--- FAIL: SerializedAttributeTest#test_serialized_attribute_with_default (unexpected)
--- FAIL: OptimisticLockingTest#test_quote_table_name (unexpected)
--- FAIL: SerializedAttributeTest#test_nil_is_always_persisted_as_null (unexpected)
--- FAIL: MultipleDbTest#test_swapping_the_connection (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_alternate_database (unexpected)
--- FAIL: DelegatedTypeTest#test_scope (unexpected)
--- FAIL: SerializedAttributeTest#test_decorated_type_with_type_for_attribute (unexpected)
--- FAIL: OptimisticLockingTest#test_non_integer_lock_destroy (unexpected)
--- FAIL: ReadOnlyTest#test_has_many_with_through_is_not_implicitly_marked_readonly_while_finding_first (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_destroy_stale_object (unexpected)
--- FAIL: OptimisticLockingTest#test_delete_with_dirty_primary_key (unexpected)
--- FAIL: ReadOnlyTest#test_cant_update_column_readonly_record (unexpected)
--- FAIL: OptimisticLockingTest#test_destroy_with_lock_version_without_default_should_work_on_dirty_value_before_type_cast (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_exception_record (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_in_base_class (unexpected)
--- FAIL: HotCompatibilityTest#test_insert_after_remove_column (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_json_attribute_returns_unserialized_value (unexpected)
--- FAIL: OptimisticLockingTest#test_update_without_attributes_does_not_only_update_lock_version (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_does_call_updated_at_when_it_is_assigned_via_a_string (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_works_under_concurrent_initial_access (unexpected)
--- FAIL: OptimisticLockingTest#test_non_integer_lock_existing (unexpected)
--- FAIL: ReadOnlyTest#test_cant_touch_readonly_record (unexpected)
--- FAIL: PrimaryKeysTest#test_find_with_more_than_one_string_key (unexpected)
--- FAIL: OptimisticLockingTest#test_polymorphic_destroy_with_dependencies_and_lock_version (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_key_format_is_not_too_precise (unexpected)
--- FAIL: PrimaryKeysTest#test_primary_key_prefix (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_primary_key_auto_assigns_on_save (unexpected)
--- FAIL: OptimisticLockingTest#test_update_with_lock_version_without_default_should_work_on_dirty_value_before_type_cast (unexpected)
--- FAIL: PrimaryKeysTest#test_primary_key_returns_value_if_it_exists (unexpected)
--- FAIL: PostgresqlBitStringTest#test_bit_string_varying_column (unexpected)
--- FAIL: ReadOnlyTest#test_find_with_readonly_option (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
--- FAIL: PostgresqlBitStringTest#test_bit_string_column (unexpected)
--- FAIL: OptimisticLockingTest#test_touch_stale_object_with_lock_without_default (unexpected)
--- FAIL: PostgresqlNumberTest#test_update (unexpected)
--- FAIL: OptimisticLockingTest#test_quote_value_passed_lock_col (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
--- FAIL: ReadOnlyTest#test_has_many_with_through_is_not_implicitly_marked_readonly_while_finding_last (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_destroy_existing_object_with_locking_column_value_null_in_the_database (unexpected)
--- FAIL: HotCompatibilityTest#test_cleans_up_after_prepared_statement_failure_in_nested_transactions (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_new_when_explicitly_passing_value (unexpected)
--- FAIL: PostgresqlInfinityTest#test_type_casting_string_on_a_float_column (unexpected)
--- FAIL: HabtmDestroyOrderTest#test_should_not_raise_error_if_have_foreign_key_in_the_join_table (unexpected)
--- FAIL: PrimaryKeysTest#test_to_key_with_default_primary_key (unexpected)
--- FAIL: PrimaryKeysTest#test_quoted_primary_key_after_set_primary_key (unexpected)
--- FAIL: InsertAllTest#test_insert_all_and_upsert_all_works_with_composite_primary_keys_when_unique_by_is_provided (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10927781?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/10927781?buildTab=artifacts#/activerecord) on master @ [c609b823b0a1fda5b7d605e4645cd452a6009961](https://github.com/cockroachdb/cockroach/commits/c609b823b0a1fda5b7d605e4645cd452a6009961):


```
--- FAIL: AssociationsJoinModelTest#test_exceptions_have_suggestions_for_fix (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_inheritance_with_custom_class_name (unexpected)
--- FAIL: BelongsToAssociationsTest#test_optional_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_selected_columns (unexpected)
--- FAIL: RelationScopingTest#test_scope_select_concatenates (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_decrement_counter_updates_lock_version (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_change_from_nil_to_persisted_record (unexpected)
--- FAIL: OptimisticLockingTest#test_counter_cache_with_touch_and_lock_version (unexpected)
--- FAIL: BelongsToAssociationsTest#test_raises_type_mismatch_with_namespaced_class (unexpected)
--- FAIL: AssociationsJoinModelTest#test_create_polymorphic_has_one_with_scope (unexpected)
--- FAIL: OptimisticLockingTest#test_touch_existing_lock (unexpected)
--- FAIL: RelationScopingTest#test_scoping_with_klass_method_works_in_the_scope_block (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_with_custom_column_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_uses_conditions_specified_on_the_has_many_association (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_change_from_persisted_record_to_new_record (unexpected)
--- FAIL: AssociationsJoinModelTest#test_eager_load_has_many_through_has_many_with_conditions (unexpected)
--- FAIL: BelongsToAssociationsTest#test_build_with_block (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_on_multiple_records (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_suffix (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_empty_update (unexpected)
--- FAIL: DelegatedTypeTest#test_association_uuid (unexpected)
--- FAIL: BelongsToAssociationsTest#test_assigning_nil_on_an_association_clears_the_associations_inverse (unexpected)
--- FAIL: BelongsToAssociationsTest#test_reloading_association_with_key_change (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_custom_select_and_order (unexpected)
--- FAIL: BelongsToAssociationsTest#test_with_select (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_works_with_model_called_Record (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_has_many_find_by_id (unexpected)
--- FAIL: BelongsToAssociationsTest#test_assignment_before_child_saved_with_primary_key (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type_cast_from_invalid_string (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_existing (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_reassign_with_namespaced_models_and_counters (unexpected)
--- FAIL: BelongsToAssociationsTest#test_assigning_belongs_to_on_destroyed_object (unexpected)
--- FAIL: BelongsToAssociationsTest#test_with_condition (unexpected)
--- FAIL: BelongsToAssociationsTest#test_multiple_counter_cache_with_after_create_update (unexpected)
--- FAIL: BelongsToAssociationsTest#test_type_mismatch (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_first (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through_with_multiple_tags (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_increment_counter_updates_custom_lock_version (unexpected)
--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object (unexpected)
--- FAIL: AssociationsJoinModelTest#test_eager_has_many_polymorphic_with_source_type (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_custom_select_and_joins (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter_after_update (unexpected)
--- FAIL: BelongsToAssociationsTest#test_reloading_the_belonging_object (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_create_model_with_inheritance (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_has_many_find_all_with_custom_class (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11045314?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11045314?buildTab=artifacts#/activerecord) on master @ [9d976cbcf5b0bf031a5d831daf47abeacdfd974d](https://github.com/cockroachdb/cockroach/commits/9d976cbcf5b0bf031a5d831daf47abeacdfd974d):


```
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_true (unexpected)
--- FAIL: CounterCacheTest#test_decrement_counter (unexpected)
--- FAIL: CounterCacheTest#test_reset_multiple_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: PreloaderTest#test_preload_does_not_group_same_class_different_scope (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_with_touch:_:written_on (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
--- FAIL: TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_parent_record (unexpected)
--- FAIL: CockroachDB::PostgresqlSerialTest#test_serial_column (unexpected)
--- FAIL: DelegatedTypeTest#test_association_uuid (unexpected)
--- FAIL: CounterCacheTest#test_the_passed_symbol_needs_to_be_an_association_name_or_counter_name (unexpected)
--- FAIL: AssociationsExtensionsTest#test_extension_name (unexpected)
--- FAIL: CounterCacheTest#test_reset_the_right_counter_if_two_have_the_same_class_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_array_types_can_be_specified (unexpected)
--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_custom_type (unexpected)
--- FAIL: AssociationsExtensionsTest#test_proxy_association_after_scoped (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_children_inherit_custom_properties (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_immutable_strings_by_default_changes_schema_inference_for_string_columns (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersMultiDbTest#test_loading_relations_with_multi_db_connection_handlers (unexpected)
--- FAIL: TimestampTest#test_saving_a_record_with_a_belongs_to_that_specifies_touching_a_specific_attribute_the_parent_should_update_that_attribute (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_with_touch:_:written_on (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_doesn't_touch_timestamps_by_default (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_defaults_are_not_touched_on_the_columns (unexpected)
--- FAIL: DelegatedTypeTest#test_delegated_type_predicates (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
--- FAIL: CounterCacheTest#test_counter_caches_are_updated_in_memory_when_the_default_value_is_nil (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_immutable_strings_by_default_retains_limit_information (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_range_types_can_be_specified (unexpected)
--- FAIL: CounterCacheTest#test_update_multiple_counters_with_touch:_true (unexpected)
--- FAIL: TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_polymorphic_parent_record_changes_within_same_class (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_user_provided_defaults_are_persisted_even_if_unchanged (unexpected)
--- FAIL: CounterCacheTest#test_reset_counters_by_counter_name (unexpected)
--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_timestamptz_if_datetime_type_changed (unexpected)
--- FAIL: DelegatedTypeTest#test_delegated_class (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_for_default_values (unexpected)
--- FAIL: DelegatedTypeTest#test_accessor (unexpected)
--- FAIL: CounterCacheTest#test_update_multiple_counters_with_touch:_%i(_updated_at_written_on_) (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_timestamp (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_are_memoized_before_type_casting (unexpected)
--- FAIL: AssociationsExtensionsTest#test_named_extension_and_block_on_habtm (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11108583?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11108583?buildTab=artifacts#/activerecord) on master @ [f295bd861a3a427652b19c2254d2401ebb4a3c8e](https://github.com/cockroachdb/cockroach/commits/f295bd861a3a427652b19c2254d2401ebb4a3c8e):


```
--- FAIL: PrimaryKeysTest#test_find_with_more_than_one_string_key (unexpected)
--- FAIL: AssociationsTest#test_subselect (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
--- FAIL: SchemaDumperTest#test_dump_schema_information_with_empty_versions (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
--- FAIL: PrimaryKeysTest#test_update_with_non_primary_key_id_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_not_set (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_schema_dump_with_default_value (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_average_interval_type (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_column (unexpected)
--- FAIL: AssociationsTest#test_eager_loading_should_not_change_count_of_children (unexpected)
--- FAIL: PrimaryKeysTest#test_integer_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_restrict_foreign_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_not_set (unexpected)
--- FAIL: PrimaryKeysTest#test_to_key_with_default_primary_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_primary_key_can_be_get_before_saving (unexpected)
--- FAIL: AssociationsTest#test_association_with_references (unexpected)
--- FAIL: CockroachDB::DefaultNumbersTest#test_default_decimal_zero_with_large_scale (unexpected)
--- FAIL: PrimaryKeysTest#test_update_counters_should_quote_pkey_and_quote_counter_columns (unexpected)
--- FAIL: AssociationsTest#test_include_with_order_works (unexpected)
--- FAIL: CockroachDB::DirtyTest#test_field_named_field (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type_cast_string_and_numeric_from_user (unexpected)
--- FAIL: CockroachDB::PrimaryKeysTest#test_serial_with_unquoted_sequence_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ColumnsTest#test_change_column_default_with_from_and_to (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_postgresql_interval_parser_error (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
--- FAIL: PrimaryKeysTest#test_delete_should_quote_pkey (unexpected)
--- FAIL: PrimaryKeysTest#test_read_attribute_with_custom_primary_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
--- FAIL: PrimaryKeysTest#test_to_key_with_customized_primary_key (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_primary_key_auto_assigns_on_save (unexpected)
--- FAIL: PrimaryKeysTest#test_quoted_primary_key_after_set_primary_key (unexpected)
--- FAIL: AssociationsTest#test_bad_collection_keys (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_column (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_string_primary_key_settable_before_save (unexpected)
--- FAIL: PrimaryKeysTest#test_assign_id_raises_error_if_primary_key_doesnt_exist (unexpected)
--- FAIL: PrimaryKeysTest#test_primary_key_prefix (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_prefix (unexpected)
--- FAIL: PrimaryKeysTest#test_primary_key_returns_nil_if_it_does_not_exist (unexpected)
--- FAIL: AssociationsTest#test_force_reload (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_interval_type_cast_from_invalid_string (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana//1690610751262/1690619566842)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11114747?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11114747?buildTab=artifacts#/activerecord) on master @ [f295bd861a3a427652b19c2254d2401ebb4a3c8e](https://github.com/cockroachdb/cockroach/commits/f295bd861a3a427652b19c2254d2401ebb4a3c8e):


```
--- FAIL: OptimisticLockingTest#test_lock_column_is_mass_assignable (unexpected)
--- FAIL: HasManyAssociationsTest#test_updates_counter_cache_when_default_scope_is_given (unexpected)
--- FAIL: HasManyAssociationsTest#test_can_rewhere_the_default_scope_of_the_associated_model (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ReferencesForeignKeyTest#test_multiple_foreign_keys_can_be_added_to_the_same_table (unexpected)
--- FAIL: HasManyAssociationsTest#test_include_method_in_has_many_association_should_return_true_for_instance_added_with_build (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: HasManyAssociationsTest#test_adding_a_collection (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_first (unexpected)
--- FAIL: HasManyAssociationsTest#test_pushing_association_updates_counter_cache (unexpected)
--- FAIL: HasManyAssociationsTest#test_association_keys_bypass_attribute_protection (unexpected)
--- FAIL: HasManyAssociationsTest#test_finding_using_primary_key (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_update_on_id_changes_the_counter_cache (unexpected)
--- FAIL: AssociationsExtensionsTest#test_marshalling_named_extensions (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_not_backed_by_database_columns_appear_in_inspect (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_increment_counter_updates_lock_version (unexpected)
--- FAIL: AssociationsExtensionsTest#test_named_extension_on_habtm (unexpected)
--- FAIL: HasManyAssociationsTest#test_replace (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_destroy_existing_object_with_locking_column_value_null_in_the_database (unexpected)
--- FAIL: LintTest#test_to_partial_path (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_with_bang_on_has_many_raises_when_record_not_saved (unexpected)
--- FAIL: HasManyAssociationsTest#test_attributes_are_set_when_initialized_from_has_many_null_relationship (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V6_0#test_create_join_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_conditions_with_record_arg (unexpected)
--- FAIL: HasManyAssociationsTest#test_new_the_association_with_an_array (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_each_with_conditions (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_with_overridden_types_keep_their_type_when_a_default_value_is_configured_separately (unexpected)
--- FAIL: HasManyAssociationsTest#test_finding_array_compatibility (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_for_default_values_are_evaluated_even_after_column_defaults_is_called (unexpected)
--- FAIL: HasManyAssociationsTest#test_dynamic_find_should_respect_association_order (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_one_on_loaded_association_should_not_use_query (unexpected)
--- FAIL: HasManyAssociationsTest#test_default_select (unexpected)
--- FAIL: HasManyAssociationsTest#test_transactions_when_replacing_on_new_record (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_by_string_id (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_straight_inheritance_uniqueness (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_many_with_merged_options (unexpected)
--- FAIL: HasManyAssociationsTest#test_prevent_double_insertion_of_new_object_when_the_parent_association_loaded_in_the_after_save_callback (unexpected)
--- FAIL: HasManyAssociationsTest#test_include_checks_if_record_exists_if_target_not_loaded (unexpected)
--- FAIL: HasManyAssociationsTest#test_clearing_updates_counter_cache_when_inverse_counter_cache_is_a_symbol_with_dependent_destroy (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_not_set (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_are_memoized_before_type_casting (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_on_empty_relation (unexpected)
--- FAIL: HasManyAssociationsTest#test_default_scope_on_relations_is_not_cached (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_from_polymorphic_association_sets_inverse_instance (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_with_custom_column_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_not_backed_by_database_columns_keep_their_type_when_a_default_value_is_configured_separately (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana//1690696936635/1690706105014)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11200326?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11200326?buildTab=artifacts#/activerecord) on master @ [695d4ec85157a4da0c97237e6f16bf919eebf6c5](https://github.com/cockroachdb/cockroach/commits/695d4ec85157a4da0c97237e6f16bf919eebf6c5):


```
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_when_integer_out_of_range (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_calls_updated_at_when_the_value_is_generated_at_create_time (unexpected)
--- FAIL: BasicsTest#test_custom_mutator (unexpected)
--- FAIL: BasicsTest#test_successful_comparison_of_like_class_records (unexpected)
--- FAIL: OptimisticLockingTest#test_quote_table_name (unexpected)
--- FAIL: CommentTest#test_comment_on_primary_key (unexpected)
--- FAIL: BasicsTest#test_attributes (unexpected)
--- FAIL: InheritanceTest#test_descends_from_active_record (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_on_existing_relation (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_with_two_has_many_associations (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_distinct_through_join_model (unexpected)
--- FAIL: HasOneAssociationsTest#test_proxy_assignment (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_with_custom_primary_key_on_has_many_source (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: BasicsTest#test_ignored_columns_not_included_in_SELECT (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_with_pluralize_table_names_false (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_with_hash_conditions (unexpected)
--- FAIL: AssociationsJoinModelTest#test_eager_load_has_many_through_has_many (unexpected)
--- FAIL: DefaultStringsTest#test_default_strings (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
--- FAIL: BasicsTest#test_clear_cache_when_setting_table_name (unexpected)
--- FAIL: BasicsTest#test_has_attribute (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_where_first_level_returns_nil (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_case_sensitive_uniqueness_with_attribute_passed_as_integer (unexpected)
--- FAIL: BasicsTest#test_preserving_time_objects_with_utc_time_conversion_to_default_timezone_local (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_repeating (unexpected)
--- FAIL: InverseBelongsToTests#test_recursive_inverse_on_recursive_model_has_many_inversing (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_create_model_with_inheritance (unexpected)
--- FAIL: BasicsTest#test_dont_clear_sequence_name_when_setting_explicitly (unexpected)
--- FAIL: OptimisticLockingTest#test_counter_cache_with_touch_and_lock_version (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_one (unexpected)
--- FAIL: BasicsTest#test_table_name_based_on_model_name (unexpected)
--- FAIL: InheritanceTest#test_new_with_abstract_class (unexpected)
--- FAIL: BasicsTest#test_preserving_time_objects_with_time_with_zone_conversion_to_default_timezone_utc (unexpected)
--- FAIL: HasOneAssociationsTest#test_exclusive_dependence (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_has_many_find_conditions (unexpected)
--- FAIL: HasOneAssociationsTest#test_association_enum_works_properly (unexpected)
--- FAIL: DefaultStringsTest#test_default_strings_containing_single_quotes (unexpected)
--- FAIL: OptimisticLockingTest#test_touch_existing_lock_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_three_levels_by_ping_pong (unexpected)
--- FAIL: HasOneAssociationsTest#test_build_association_dont_create_transaction (unexpected)
--- FAIL: OptimisticLockingTest#test_explicit_update_lock_column_raise_error (unexpected)
--- FAIL: InheritanceTest#test_compute_type_argument_error (unexpected)
--- FAIL: CounterCacheTest#test_reset_counter_of_has_many_:through_association (unexpected)
--- FAIL: CounterCacheTest#test_update_counters_in_a_polymorphic_relationship (unexpected)
--- FAIL: AssociationsExtensionsTest#test_marshalling_extensions (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana/teamcity-11200326-1691214430-17-n1cpu4/1691215531863/1691225006298)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #99620 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11308685?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11308685?buildTab=artifacts#/activerecord) on master @ [c13bf7633cbb416d9e43f8c57b1e309fab1110ce](https://github.com/cockroachdb/cockroach/commits/c13bf7633cbb416d9e43f8c57b1e309fab1110ce):


```
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_on_empty_relation (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_are_memoized_before_type_casting (unexpected)
--- FAIL: FixturesTest#test_erb_in_fixtures (unexpected)
--- FAIL: FixturesTest#test_attributes (unexpected)
--- FAIL: TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_polymorphic_parent_record_changes_within_same_class (unexpected)
--- FAIL: SerializedAttributeTest#test_decorated_type_with_decorator_block (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_nonexistent_attribute (unexpected)
--- FAIL: CommentTest#test_comment_on_primary_key (unexpected)
--- FAIL: SerializedAttributeTest#test_serialize_with_coder (unexpected)
--- FAIL: FixturesTest#test_logger_level_invariant (unexpected)
--- FAIL: TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_parent_record (unexpected)
--- FAIL: AttributeMethodsTest#test_attribute_present_with_booleans (unexpected)
--- FAIL: SerializedAttributeTest#test_nil_is_not_changed_when_serialized_with_a_class (unexpected)
--- FAIL: FixturesTest#test_no_args_record_returns_all_without_array (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_time_attribute (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_object_arg (unexpected)
--- FAIL: ActiveRecord::Encryption::EncryptableRecordTest#test_encrypts_store_attributes_with_accessors (unexpected)
--- FAIL: TestNestedAttributesForDelegatedType#test_should_build_a_new_record_based_on_the_delegated_type (unexpected)
--- FAIL: UniquenessValidationTest#test_validates_uniqueness_with_nil_value (unexpected)
--- FAIL: SuppressorTest#test_resumes_saving_after_suppression_complete (unexpected)
--- FAIL: BooleanTest#test_boolean_without_questionmark (unexpected)
--- FAIL: SerializedAttributeTest#test_json_read_legacy_null (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_limit (unexpected)
--- FAIL: SuppressorTest#test_suppresses_when_nested_multiple_times (unexpected)
--- FAIL: FixturesTest#test_fixtures_are_set_up_with_database_env_variable (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: SuppressorTest#test_suppresses_create (unexpected)
--- FAIL: CommentTest#test_blank_columns_created_in_block (unexpected)
--- FAIL: FixturesTest#test_no_args_returns_all (unexpected)
--- FAIL: SerializedAttributeTest#test_where_by_serialized_attribute_with_array (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_overloading_types (unexpected)
--- FAIL: FixturesTest#test_broken_yaml_exception (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_when_integer_out_of_range (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_by_default_database_collation (unexpected)
--- FAIL: FixturesTest#test_dirty_dirty_yaml_file (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_immutable_strings_by_default_does_not_affect_``attribute_:foo,_:string`` (unexpected)
--- FAIL: SerializedAttributeTest#test_unexpected_serialized_type (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_on_existing_relation (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_boolean_value_true (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_scope_array (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_aliases (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_column_should_unserialize_after_update_column (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_overloaded_properties_with_limit (unexpected)
--- FAIL: SerializedAttributeTest#test_serialized_attribute_calling_dup_method (unexpected)
--- FAIL: SuppressorTest#test_suppresses_create_in_callback (unexpected)
--- FAIL: SerializedAttributeTest#test_nil_serialized_attribute_without_class_constraint (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana/teamcity-11308685-1691992291-17-n1cpu4/1691993151134/1692001243196)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #108654 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1.9-rc release-blocker]
- #99620 roachtest: activerecord failed (waiting for upstream updates) [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11354747?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/11354747?buildTab=artifacts#/activerecord) on master @ [673bde530075ed690cf115aa0a5bdd7191d63f54](https://github.com/cockroachdb/cockroach/commits/673bde530075ed690cf115aa0a5bdd7191d63f54):


```
--- FAIL: PreloaderTest#test_some_already_loaded_associations (unexpected)
--- FAIL: EagerAssociationTest#test_preloading_with_has_one_through_an_sti_with_after_initialize (unexpected)
--- FAIL: EagerAssociationTest#test_type_cast_in_where_references_association_name (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_queries_with_offset_which_return_0_rows (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_floating_point_numbers (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_a_relation_having_distinct_and_order (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_conditions_string_with_quoted_table_name (unexpected)
--- FAIL: AssociationProxyTest#test_inverses_get_set_of_subsets_of_the_association (unexpected)
--- FAIL: EagerAssociationTest#test_with_two_tables_in_from_without_getting_double_quoted (unexpected)
--- FAIL: EagerAssociationTest#test_preconfigured_includes_with_has_many (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_has_many_and_limit_with_no_results (unexpected)
--- FAIL: EagerAssociationTest#test_preloading_has_many_through_with_distinct (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_limit_and_conditions (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_updating_attributes_on_non_rich_associations (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_explicit_join (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_conditions_hash (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation_with_includes (unexpected)
--- FAIL: EagerAssociationTest#test_include_has_many_using_primary_key (unexpected)
--- FAIL: AssociationProxyTest#test_pluck_uses_loaded_target (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
--- FAIL: EagerAssociationTest#test_order_on_join_table_with_include_and_limit (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_with_validate_false_does_not_run_associated_validation_callbacks_on_update (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_join_model_with_conditions_on_top_level (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_proper_usage_of_primary_keys_and_join_table (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_has_one_dependent_does_not_destroy_dependent (unexpected)
--- FAIL: EagerAssociationTest#test_count_with_include (unexpected)
--- FAIL: MultipleDbTest#test_transactions_across_databases (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_order_on_nested_association (unexpected)
--- FAIL: EagerAssociationTest#test_eager-loading_non-readonly_association (unexpected)
--- FAIL: EagerAssociationTest#test_exceptions_have_suggestions_for_fix (unexpected)
--- FAIL: EagerAssociationTest#test_polymorphic_type_condition (unexpected)
--- FAIL: DDLTest#test_add_geometry_column_null_false (unexpected)
--- FAIL: EagerAssociationTest#test_include_has_one_using_primary_key (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_should_property_quote_string_primary_keys (unexpected)
--- FAIL: CockroachDB::PostgresqlSerialTest#test_not_serial_column (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_count (unexpected)
--- FAIL: CockroachDB::PostgresqlSerialTest#test_serial_column (unexpected)
--- FAIL: DDLTest#test_create_simple_geography_using_shortcut (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_removing_associations_on_destroy (unexpected)
--- FAIL: EagerAssociationTest#test_has_many_through_with_order (unexpected)
--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_conditions_on_association (unexpected)
For a full summary look at the activerecord artifacts 
An updated blocklist (activeRecordBlocklist) is available in the artifacts' activerecord log
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/p/roachfana/teamcity-11354747-1692251272-17-n1cpu4/1692255224169/1692264516940)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #108654 roachtest: activerecord failed (waiting for upstream updates) [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1.9-rc]
- #99620 roachtest: activerecord failed (waiting for upstream updates) [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-23.1]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-22.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -229,9 +229,14 @@ func registerActiveRecord(r registry.Registry) {
 				results.failExpectedCount++
 				results.currentFailures = append(results.currentFailures, test)
 			case !pass && !expectedFailure:
-				results.results[test] = fmt.Sprintf("--- FAIL: %s (unexpected)", test)
-				results.failUnexpectedCount++
-				results.currentFailures = append(results.currentFailures, test)
+				// The test suite is flaky and work is being done upstream to stabilize
+				// it (https://github.com/cockroachdb/cockroach/issues/108938). Until
+				// that's done, we ignore all failures from this test.
+				// results.results[test] = fmt.Sprintf("--- FAIL: %s (unexpected)", test)
+				// results.failUnexpectedCount++
+				// results.currentFailures = append(results.currentFailures, test)
+				results.results[test] = fmt.Sprintf("--- SKIP: %s due to upstream flakes (https://github.com/cockroachdb/cockroach/issues/108938)", test)
+				results.ignoredCount++
 			}
 			results.runTests[test] = struct{}{}
 		}

--- a/pkg/cmd/roachtest/tests/sequelize.go
+++ b/pkg/cmd/roachtest/tests/sequelize.go
@@ -145,7 +145,11 @@ func registerSequelize(r registry.Registry) {
 		rawResultsStr := result.Stdout + result.Stderr
 		t.L().Printf("Test Results: %s", rawResultsStr)
 		if err != nil {
-			t.Fatal(err)
+			// The test suite is flaky and work is being done upstream to stabilize
+			// it (https://github.com/sequelize/sequelize/pull/15569). Until that's
+			// done, we ignore all failures from this test.
+			// t.Fatal(err)
+			t.L().Printf("ignoring failure (https://github.com/cockroachdb/cockroach/issues/108937): %s", err)
 		}
 	}
 

